<!DOCTYPE html>
<html><head><title>ErrorRecord</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">2 types derived from ErrorRecord</div><div class="rA">System.Management.Automation (2)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">engine\ErrorPackage.cs (1)</div>
<a href="../engine/ErrorPackage.cs.html#1714"><b>1714</b>internal class ErrorRecord&lt;TException&gt; : <i>ErrorRecord</i> where TException : Exception</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\RemotingErrorRecord.cs (1)</div>
<a href="../engine/remoting/client/RemotingErrorRecord.cs.html#15"><b>15</b>public class RemotingErrorRecord : <i>ErrorRecord</i></a>
</div>
</div>
<div class="rH">137 instantiations of ErrorRecord</div><div class="rA">System.Management.Automation (137)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">engine\ErrorPackage.cs (2)</div>
<a href="../engine/ErrorPackage.cs.html#1284"><b>1284</b>ErrorRecord er = new <i>ErrorRecord</i>();</a>
<a href="../engine/ErrorPackage.cs.html#1445"><b>1445</b>return new <i>ErrorRecord</i>(this, replaceParentContainsErrorRecordException);</a>
</div>
<div class="rF"><div class="rN">engine\ExecutionContext.cs (1)</div>
<a href="../engine/ExecutionContext.cs.html#1472"><b>1472</b>: new <i>ErrorRecord</i>(e, errorId, ErrorCategory.OperationStopped, null);</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\History.cs (13)</div>
<a href="../engine/hostifaces/History.cs.html#938"><b>938</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#959"><b>959</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1182"><b>1182</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1222"><b>1222</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1245"><b>1245</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1268"><b>1268</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1483"><b>1483</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1667"><b>1667</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1700"><b>1700</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1721"><b>1721</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1777"><b>1777</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1807"><b>1807</b>new <i>ErrorRecord</i></a>
<a href="../engine/hostifaces/History.cs.html#1849"><b>1849</b>new <i>ErrorRecord</i></a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\PowerShell.cs (2)</div>
<a href="../engine/hostifaces/PowerShell.cs.html#3663"><b>3663</b>this.Streams.Error.Add(new <i>ErrorRecord</i>(e,</a>
<a href="../engine/hostifaces/PowerShell.cs.html#4764"><b>4764</b>this.Streams.Error.Add(new <i>ErrorRecord</i>(e,</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\PSTask.cs (1)</div>
<a href="../engine/hostifaces/PSTask.cs.html#157"><b>157</b>var errorRecord = new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">engine\InternalCommands.cs (2)</div>
<a href="../engine/InternalCommands.cs.html#784"><b>784</b>WriteError(new <i>ErrorRecord</i>(ex, &quot;MethodInvocationError&quot;, ErrorCategory.InvalidOperation, _inputObject));</a>
<a href="../engine/InternalCommands.cs.html#1090"><b>1090</b>WriteError(new <i>ErrorRecord</i>(ex, &quot;MethodInvocationError&quot;, ErrorCategory.InvalidOperation, _inputObject));</a>
</div>
<div class="rF"><div class="rN">engine\Modules\ModuleCmdletBase.cs (1)</div>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#2444"><b>2444</b>ErrorRecord er = new <i>ErrorRecord</i>(e, &quot;FormatXmlUpdateException&quot;, ErrorCategory.InvalidOperation,</a>
</div>
<div class="rF"><div class="rN">engine\Modules\RemoteDiscoveryHelper.cs (4)</div>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#339"><b>339</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(outerException, DiscoveryProviderNotFoundErrorId, ErrorCategory.NotImplemented, null);</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#349"><b>349</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(outerException, &quot;DiscoveryProviderFailure&quot;, ErrorCategory.NotSpecified, null);</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#366"><b>366</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(outerException, errorId, errorCategory, null);</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#392"><b>392</b>ErrorRecord outerErrorRecord = new <i>ErrorRecord</i>(innerErrorRecord, null /* null means: do not replace the exception */);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\ClientMethodExecutor.cs (1)</div>
<a href="../engine/remoting/client/ClientMethodExecutor.cs.html#234"><b>234</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\Job.cs (5)</div>
<a href="../engine/remoting/client/Job.cs.html#2075"><b>2075</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;PSJobConnectFailed&quot;, ErrorCategory.InvalidOperation, _psRemoteChildJob);</a>
<a href="../engine/remoting/client/Job.cs.html#2261"><b>2261</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;StopJobCannotConnectToServer&quot;,</a>
<a href="../engine/remoting/client/Job.cs.html#3321"><b>3321</b>failureErrorRecord = new <i>ErrorRecord</i>(helper.InternalException,</a>
<a href="../engine/remoting/client/Job.cs.html#3382"><b>3382</b>failureErrorRecord = new <i>ErrorRecord</i>(failureException, targetObject,</a>
<a href="../engine/remoting/client/Job.cs.html#3420"><b>3420</b>errorRecord = new <i>ErrorRecord</i>(pipeline.PipelineStateInfo.Reason,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\Job2.cs (5)</div>
<a href="../engine/remoting/client/Job2.cs.html#814"><b>814</b>ExecutionError.Add(new <i>ErrorRecord</i>(e, &quot;ContainerParentJobStartError&quot;,</a>
<a href="../engine/remoting/client/Job2.cs.html#999"><b>999</b>ExecutionError.Add(new <i>ErrorRecord</i>(e, &quot;ContainerParentJobResumeError&quot;,</a>
<a href="../engine/remoting/client/Job2.cs.html#1238"><b>1238</b>ExecutionError.Add(new <i>ErrorRecord</i>(e, &quot;ContainerParentJobUnblockError&quot;,</a>
<a href="../engine/remoting/client/Job2.cs.html#1410"><b>1410</b>ExecutionError.Add(new <i>ErrorRecord</i>(e, &quot;ContainerParentJobSuspendError&quot;,</a>
<a href="../engine/remoting/client/Job2.cs.html#1586"><b>1586</b>ExecutionError.Add(new <i>ErrorRecord</i>(e, &quot;ContainerParentJobStopError&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\JobManager.cs (1)</div>
<a href="../engine/remoting/client/JobManager.cs.html#847"><b>847</b>cmdlet.WriteError(new <i>ErrorRecord</i>(exception, identifier, ErrorCategory.OpenError, sourceAdapter));</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\remoterunspace.cs (1)</div>
<a href="../engine/remoting/client/remoterunspace.cs.html#1965"><b>1965</b>new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ConnectPSSession.cs (3)</div>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#742"><b>742</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, FQEID, ErrorCategory.InvalidOperation, null);</a>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#866"><b>866</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;CannotConnectVMContainerSession&quot;, ErrorCategory.InvalidOperation, psSession);</a>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#889"><b>889</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;PSSessionConnectFailed&quot;, ErrorCategory.InvalidOperation, psSession);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\CustomShellCommands.cs (1)</div>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#5267"><b>5267</b>ThrowTerminatingError(new <i>ErrorRecord</i>(e.ErrorRecord.Exception, &quot;CouldNotFindSessionConfiguration&quot;, ErrorCategory.ObjectNotFound, this.ConfigurationName));</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\DisconnectPSSession.cs (3)</div>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#217"><b>217</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;CannotDisconnectVMContainerSession&quot;, ErrorCategory.InvalidOperation, psSession);</a>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#246"><b>246</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;CannotDisconnectSessionWhenNotOpened&quot;, ErrorCategory.InvalidOperation, psSession);</a>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#505"><b>505</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;PSSessionDisconnectFailed&quot;, ErrorCategory.InvalidOperation, _remoteSession);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\EnterPSHostProcessCommand.cs (1)</div>
<a href="../engine/remoting/commands/EnterPSHostProcessCommand.cs.html#201"><b>201</b>new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\newrunspacecommand.cs (3)</div>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#541"><b>541</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#600"><b>600</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#1058"><b>1058</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(e,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\PSRemotingCmdlet.cs (3)</div>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#1850"><b>1850</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(e,</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#3921"><b>3921</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, FQEID, ErrorCategory.InvalidOperation, connectionInfo);</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#4047"><b>4047</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;PSSessionNameMatchFail&quot;, ErrorCategory.InvalidOperation, name);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\PushRunspaceCommand.cs (3)</div>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#627"><b>627</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(exception, argument,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1072"><b>1072</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(e,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1248"><b>1248</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(e,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ReceiveJob.cs (1)</div>
<a href="../engine/remoting/commands/ReceiveJob.cs.html#907"><b>907</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(resultReason, &quot;JobStateFailed&quot;, ErrorCategory.InvalidResult, null);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ReceivePSSession.cs (2)</div>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#447"><b>447</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, FQEID, ErrorCategory.InvalidOperation, connectionInfo);</a>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#702"><b>702</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;CannotReceiveVMContainerSession&quot;, ErrorCategory.InvalidOperation, session);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\RemoveJob.cs (4)</div>
<a href="../engine/remoting/commands/RemoveJob.cs.html#100"><b>100</b>WriteError(new <i>ErrorRecord</i>(ex, &quot;JobWithSpecifiedNameNotFound&quot;, ErrorCategory.ObjectNotFound, name));</a>
<a href="../engine/remoting/commands/RemoveJob.cs.html#115"><b>115</b>WriteError(new <i>ErrorRecord</i>(ex, &quot;JobObjectNotFinishedCannotBeRemoved&quot;, ErrorCategory.InvalidOperation, job2));</a>
<a href="../engine/remoting/commands/RemoveJob.cs.html#337"><b>337</b>WriteError(new <i>ErrorRecord</i>(ex, &quot;JobWithSpecifiedSessionNotFound&quot;, ErrorCategory.ObjectNotFound, id));</a>
<a href="../engine/remoting/commands/RemoveJob.cs.html#596"><b>596</b>WriteError(new <i>ErrorRecord</i>(ex, &quot;JobObjectNotFinishedCannotBeRemoved&quot;, ErrorCategory.InvalidOperation, job));</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\removerunspacecommand.cs (1)</div>
<a href="../engine/remoting/commands/removerunspacecommand.cs.html#161"><b>161</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(reason, &quot;RemoveSessionCannotConnectToServer&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\WaitJob.cs (1)</div>
<a href="../engine/remoting/commands/WaitJob.cs.html#378"><b>378</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\common\WireDataFormat\EncodeAndDecode.cs (2)</div>
<a href="../engine/remoting/common/WireDataFormat/EncodeAndDecode.cs.html#1556"><b>1556</b>er = new <i>ErrorRecord</i>(er, exception);</a>
<a href="../engine/remoting/common/WireDataFormat/EncodeAndDecode.cs.html#1577"><b>1577</b>new <i>ErrorRecord</i>(exception, errorId, category, null);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\common\WireDataFormat\RemoteHostEncoder.cs (2)</div>
<a href="../engine/remoting/common/WireDataFormat/RemoteHostEncoder.cs.html#263"><b>263</b>errorRecord = new <i>ErrorRecord</i>(exception, &quot;RemoteHostExecutionException&quot;, ErrorCategory.NotSpecified, null);</a>
<a href="../engine/remoting/common/WireDataFormat/RemoteHostEncoder.cs.html#270"><b>270</b>errorRecord = new <i>ErrorRecord</i>(errorRecord, exception);</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Operations\MiscOps.cs (3)</div>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1538"><b>1538</b>var errorRecord = new <i>ErrorRecord</i>(current.ErrorRecordToPass, current.ExceptionToPass);</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1743"><b>1743</b>locals.SetAutomaticVariable(AutomaticVariable.Underbar, new <i>ErrorRecord</i>(err, exception), context);</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2015"><b>2015</b>er = new <i>ErrorRecord</i>(exception, message, ErrorCategory.OperationStopped, null);</a>
</div>
<div class="rF"><div class="rN">engine\SessionStateContainer.cs (2)</div>
<a href="../engine/SessionStateContainer.cs.html#4354"><b>4354</b>context.WriteError(new <i>ErrorRecord</i>(</a>
<a href="../engine/SessionStateContainer.cs.html#4397"><b>4397</b>context.WriteError(new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">engine\SessionStateNavigation.cs (1)</div>
<a href="../engine/SessionStateNavigation.cs.html#550"><b>550</b>context.WriteError(new <i>ErrorRecord</i>(e, &quot;NormalizePathNullResult&quot;, ErrorCategory.InvalidArgument, path));</a>
</div>
<div class="rF"><div class="rN">engine\SessionStateProviderAPIs.cs (1)</div>
<a href="../engine/SessionStateProviderAPIs.cs.html#1488"><b>1488</b>new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\FormattingObjectsDeserializer.cs (1)</div>
<a href="../FormatAndOutput/common/FormattingObjectsDeserializer.cs.html#472"><b>472</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\FormatViewManager.cs (2)</div>
<a href="../FormatAndOutput/common/FormatViewManager.cs.html#652"><b>652</b>errorRecord = new <i>ErrorRecord</i>(</a>
<a href="../FormatAndOutput/common/FormatViewManager.cs.html#666"><b>666</b>errorRecord = new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">help\HelpCommands.cs (3)</div>
<a href="../help/HelpCommands.cs.html#385"><b>385</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(e, &quot;InvalidHelpCategory&quot;, ErrorCategory.InvalidArgument, null);</a>
<a href="../help/HelpCommands.cs.html#493"><b>493</b>WriteError(new <i>ErrorRecord</i>(innerException, &quot;NoParmsFound&quot;, ErrorCategory.InvalidArgument, helpInfo));</a>
<a href="../help/HelpCommands.cs.html#716"><b>716</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(e, &quot;ParameterNotValidInRemoteRunspace&quot;, ErrorCategory.InvalidArgument, null);</a>
</div>
<div class="rF"><div class="rN">help\HelpProvider.cs (1)</div>
<a href="../help/HelpProvider.cs.html#213"><b>213</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(exception, &quot;LoadHelpFileForTargetFailed&quot;, ErrorCategory.OpenError, null);</a>
</div>
<div class="rF"><div class="rN">help\HelpSystem.cs (1)</div>
<a href="../help/HelpSystem.cs.html#334"><b>334</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(e, &quot;HelpNotFound&quot;, ErrorCategory.ResourceUnavailable, null);</a>
</div>
<div class="rF"><div class="rN">help\UpdatableHelpCommandBase.cs (1)</div>
<a href="../help/UpdatableHelpCommandBase.cs.html#624"><b>624</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(new Exception(errorMessage),</a>
</div>
<div class="rF"><div class="rN">help\UpdatableHelpSystem.cs (1)</div>
<a href="../help/UpdatableHelpSystem.cs.html#118"><b>118</b>return new <i>ErrorRecord</i>(new Exception(GetExceptionMessage(commandType)), Exception.FullyQualifiedErrorId, Exception.ErrorCategory,</a>
</div>
<div class="rF"><div class="rN">namespaces\FileSystemContentStream.cs (2)</div>
<a href="../namespaces/FileSystemContentStream.cs.html#417"><b>417</b>_provider.WriteError(new <i>ErrorRecord</i>(e, &quot;GetContentReaderIOError&quot;, ErrorCategory.ReadError, _path));</a>
<a href="../namespaces/FileSystemContentStream.cs.html#571"><b>571</b>_provider.WriteError(new <i>ErrorRecord</i>(e, &quot;GetContentReaderIOError&quot;, ErrorCategory.ReadError, _path));</a>
</div>
<div class="rF"><div class="rN">namespaces\FileSystemProvider.cs (37)</div>
<a href="../namespaces/FileSystemProvider.cs.html#578"><b>578</b>WriteError(new <i>ErrorRecord</i>(e, &quot;DriveRootError&quot;, ErrorCategory.ReadError, drive));</a>
<a href="../namespaces/FileSystemProvider.cs.html#1334"><b>1334</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#1353"><b>1353</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#1712"><b>1712</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#2426"><b>2426</b>WriteError(new <i>ErrorRecord</i>(e, &quot;AccessException&quot;, ErrorCategory.PermissionDenied, strTargetPath));</a>
<a href="../namespaces/FileSystemProvider.cs.html#2457"><b>2457</b>WriteError(new <i>ErrorRecord</i>(e, &quot;AccessException&quot;, ErrorCategory.PermissionDenied, path));</a>
<a href="../namespaces/FileSystemProvider.cs.html#2486"><b>2486</b>WriteError(new <i>ErrorRecord</i>(exception, &quot;NewItemDeleteIOError&quot;, ErrorCategory.WriteError, path));</a>
<a href="../namespaces/FileSystemProvider.cs.html#2592"><b>2592</b>WriteError(new <i>ErrorRecord</i>(e, &quot;AccessException&quot;, ErrorCategory.PermissionDenied, strTargetPath));</a>
<a href="../namespaces/FileSystemProvider.cs.html#2620"><b>2620</b>WriteError(new <i>ErrorRecord</i>(e, &quot;AccessException&quot;, ErrorCategory.PermissionDenied, strTargetPath));</a>
<a href="../namespaces/FileSystemProvider.cs.html#2657"><b>2657</b>WriteError(new <i>ErrorRecord</i>(exception, &quot;NewItemDeleteIOError&quot;, ErrorCategory.WriteError, path));</a>
<a href="../namespaces/FileSystemProvider.cs.html#2707"><b>2707</b>WriteError(new <i>ErrorRecord</i>(exception, &quot;NewItemCreateIOError&quot;, ErrorCategory.WriteError, path));</a>
<a href="../namespaces/FileSystemProvider.cs.html#2820"><b>2820</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#3011"><b>3011</b>WriteError(new <i>ErrorRecord</i>(e, &quot;ItemDoesNotExist&quot;, ErrorCategory.ObjectNotFound, path));</a>
<a href="../namespaces/FileSystemProvider.cs.html#3062"><b>3062</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#3688"><b>3688</b>WriteError(new <i>ErrorRecord</i>(e, &quot;CopyError&quot;, ErrorCategory.WriteError, path));</a>
<a href="../namespaces/FileSystemProvider.cs.html#3735"><b>3735</b>WriteError(new <i>ErrorRecord</i>(e, &quot;CopyItemRemotelyFailedToReadFile&quot;, ErrorCategory.WriteError, path));</a>
<a href="../namespaces/FileSystemProvider.cs.html#3765"><b>3765</b>WriteError(new <i>ErrorRecord</i>(e, &quot;CopyError&quot;, ErrorCategory.WriteError, destinationPath));</a>
<a href="../namespaces/FileSystemProvider.cs.html#3882"><b>3882</b>WriteError(new <i>ErrorRecord</i>(e, &quot;CopyError&quot;, ErrorCategory.WriteError, destination));</a>
<a href="../namespaces/FileSystemProvider.cs.html#3993"><b>3993</b>WriteError(new <i>ErrorRecord</i>(e, &quot;CopyError&quot;, ErrorCategory.WriteError, destinationPath));</a>
<a href="../namespaces/FileSystemProvider.cs.html#4132"><b>4132</b>WriteError(new <i>ErrorRecord</i>(e, &quot;CopyError&quot;, ErrorCategory.WriteError, sourceDirectoryFullName));</a>
<a href="../namespaces/FileSystemProvider.cs.html#4450"><b>4450</b>WriteError(new <i>ErrorRecord</i>(e, &quot;FailedToCopyFileFromRemoteSession&quot;, ErrorCategory.WriteError, sourceFileFullName));</a>
<a href="../namespaces/FileSystemProvider.cs.html#4847"><b>4847</b>WriteError(new <i>ErrorRecord</i>(e, &quot;RemotePathNotFound&quot;, ErrorCategory.WriteError, destinationPath));</a>
<a href="../namespaces/FileSystemProvider.cs.html#4893"><b>4893</b>WriteError(new <i>ErrorRecord</i>(e, &quot;CopyError&quot;, ErrorCategory.WriteError, destination));</a>
<a href="../namespaces/FileSystemProvider.cs.html#4924"><b>4924</b>WriteError(new <i>ErrorRecord</i>(e, &quot;FailedToCreateDirectory&quot;, ErrorCategory.WriteError, destination));</a>
<a href="../namespaces/FileSystemProvider.cs.html#4933"><b>4933</b>WriteError(new <i>ErrorRecord</i>(e, &quot;DirectoryExist&quot;, ErrorCategory.ResourceExists, path));</a>
<a href="../namespaces/FileSystemProvider.cs.html#4949"><b>4949</b>WriteError(new <i>ErrorRecord</i>(e, errorId, ErrorCategory.WriteError, destinationPath));</a>
<a href="../namespaces/FileSystemProvider.cs.html#5215"><b>5215</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#5235"><b>5235</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#6144"><b>6144</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#6203"><b>6203</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#6244"><b>6244</b>WriteError(new <i>ErrorRecord</i>(e, &quot;GetValueError&quot;, ErrorCategory.ReadError, property));</a>
<a href="../namespaces/FileSystemProvider.cs.html#6409"><b>6409</b>WriteError(new <i>ErrorRecord</i>(e, &quot;SetPropertyError&quot;, ErrorCategory.ReadError, property));</a>
<a href="../namespaces/FileSystemProvider.cs.html#6425"><b>6425</b>WriteError(new <i>ErrorRecord</i>(e, &quot;SetPropertyError&quot;, ErrorCategory.ReadError, property));</a>
<a href="../namespaces/FileSystemProvider.cs.html#6439"><b>6439</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#6707"><b>6707</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/FileSystemProvider.cs.html#7486"><b>7486</b>fileSystemContext.WriteError(new <i>ErrorRecord</i>(e, &quot;CopyFileRemoteExecutionError&quot;, ErrorCategory.InvalidOperation, ps));</a>
<a href="../namespaces/FileSystemProvider.cs.html#7491"><b>7491</b>cmdletContext.WriteError(new <i>ErrorRecord</i>(e, &quot;CopyFileRemoteExecutionError&quot;, ErrorCategory.InvalidOperation, ps));</a>
</div>
<div class="rF"><div class="rN">namespaces\SessionStateProviderBase.cs (4)</div>
<a href="../namespaces/SessionStateProviderBase.cs.html#612"><b>612</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/SessionStateProviderBase.cs.html#686"><b>686</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/SessionStateProviderBase.cs.html#763"><b>763</b>WriteError(new <i>ErrorRecord</i>(</a>
<a href="../namespaces/SessionStateProviderBase.cs.html#868"><b>868</b>WriteError(new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">security\CatalogHelper.cs (2)</div>
<a href="../security/CatalogHelper.cs.html#111"><b>111</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(exception, &quot;UnKnownCatalogVersion&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#142"><b>142</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(exception, &quot;UnKnownCatalogVersion&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">utils\ExecutionExceptions.cs (1)</div>
<a href="../utils/ExecutionExceptions.cs.html#67"><b>67</b>_errorRecord = new <i>ErrorRecord</i>(icer.ErrorRecord, innerException);</a>
</div>
<div class="rF"><div class="rN">utils\PathUtils.cs (1)</div>
<a href="../utils/PathUtils.cs.html#180"><b>180</b>ErrorRecord errorRecord = new <i>ErrorRecord</i>(</a>
</div>
<div class="rF"><div class="rN">utils\SessionStateExceptions.cs (4)</div>
<a href="../utils/SessionStateExceptions.cs.html#72"><b>72</b>_errorRecord = new <i>ErrorRecord</i>(icer.ErrorRecord, innerException);</a>
<a href="../utils/SessionStateExceptions.cs.html#76"><b>76</b>_errorRecord = new <i>ErrorRecord</i>(</a>
<a href="../utils/SessionStateExceptions.cs.html#208"><b>208</b>_errorRecord = new <i>ErrorRecord</i>(icer.ErrorRecord, errorRecordException);</a>
<a href="../utils/SessionStateExceptions.cs.html#212"><b>212</b>_errorRecord = new <i>ErrorRecord</i>(</a>
</div>
</div>
<div class="rH">879 references to ErrorRecord</div><div class="rA">Microsoft.Management.Infrastructure.CimCmdlets (33)</div><div class="rG" id="Microsoft.Management.Infrastructure.CimCmdlets"><div class="rF"><div class="rN">CimCommandBase.cs (1)</div>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimCommandBase.cs.html#678"><b>678</b><i>ErrorRecord</i> errorRecord = new(exception, operation, ErrorCategory.InvalidOperation, this);</a>
</div>
<div class="rF"><div class="rN">CimRegisterCimIndication.cs (1)</div>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimRegisterCimIndication.cs.html#269"><b>269</b><i>ErrorRecord</i> errorRecord = ErrorToErrorRecord.ErrorRecordFromAnyException(</a>
</div>
<div class="rF"><div class="rN">CimSessionOperations.cs (15)</div>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#508"><b>508</b>ref List&lt;<i>ErrorRecord</i>&gt; errRecords,</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#527"><b>527</b>out IEnumerable&lt;<i>ErrorRecord</i>&gt; errorRecords)</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#531"><b>531</b>List&lt;<i>ErrorRecord</i>&gt; errRecords = new();</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#560"><b>560</b>out IEnumerable&lt;<i>ErrorRecord</i>&gt; errorRecords)</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#564"><b>564</b>List&lt;<i>ErrorRecord</i>&gt; errRecords = new();</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#592"><b>592</b>out IEnumerable&lt;<i>ErrorRecord</i>&gt; errorRecords)</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#596"><b>596</b>List&lt;<i>ErrorRecord</i>&gt; errRecords = new();</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#635"><b>635</b>out IEnumerable&lt;<i>ErrorRecord</i>&gt; errorRecords)</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#639"><b>639</b>List&lt;<i>ErrorRecord</i>&gt; errRecords = new();</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#673"><b>673</b>out IEnumerable&lt;<i>ErrorRecord</i>&gt; errorRecords)</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#677"><b>677</b>List&lt;<i>ErrorRecord</i>&gt; errRecords = new();</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#1087"><b>1087</b>IEnumerable&lt;<i>ErrorRecord</i>&gt; errorRecords = null;</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#1124"><b>1124</b>foreach (<i>ErrorRecord</i> errRecord in errorRecords)</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#1169"><b>1169</b>IEnumerable&lt;<i>ErrorRecord</i>&gt; errorRecords = null;</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs.html#1204"><b>1204</b>foreach (<i>ErrorRecord</i> errRecord in errorRecords)</a>
</div>
<div class="rF"><div class="rN">CimWriteError.cs (13)</div>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#20"><b>20</b>/// Convert error or exception to &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#35"><b>35</b>internal static <i>ErrorRecord</i> ErrorRecordFromAnyException(</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#69"><b>69</b>/// Create &lt;see cref=&quot;<i>ErrorRecord</i>&quot;/&gt; from &lt;see cref=&quot;CimException&quot;/&gt; object.</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#75"><b>75</b>internal static <i>ErrorRecord</i> CreateFromCimException(</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#86"><b>86</b>/// Create &lt;see cref=&quot;<i>ErrorRecord</i>&quot;/&gt; from &lt;see cref=&quot;Exception&quot;/&gt; object.</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#94"><b>94</b>internal static <i>ErrorRecord</i> InitializeErrorRecord(</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#110"><b>110</b>/// Create &lt;see cref=&quot;<i>ErrorRecord</i>&quot;/&gt; from &lt;see cref=&quot;CimException&quot;/&gt; object.</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#116"><b>116</b>internal static <i>ErrorRecord</i> InitializeErrorRecord(</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#121"><b>121</b><i>ErrorRecord</i> errorRecord = InitializeErrorRecordCore(</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#137"><b>137</b>/// Create &lt;see cref=&quot;<i>ErrorRecord</i>&quot;/&gt; from &lt;see cref=&quot;Exception&quot;/&gt; object.</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#145"><b>145</b>internal static <i>ErrorRecord</i> InitializeErrorRecordCore(</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#169"><b>169</b><i>ErrorRecord</i> coreErrorRecord = new(</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CimWriteError.cs.html#184"><b>184</b><i>ErrorRecord</i> errorRecord = new System.Management.Automation.Runspaces.RemotingErrorRecord(</a>
</div>
<div class="rF"><div class="rN">CmdletOperation.cs (3)</div>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CmdletOperation.cs.html#69"><b>69</b>public virtual void ThrowTerminatingError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CmdletOperation.cs.html#84"><b>84</b>public virtual void WriteError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../../Microsoft.Management.Infrastructure.CimCmdlets/CmdletOperation.cs.html#122"><b>122</b><i>ErrorRecord</i> errorRecord = new(exception, operation, ErrorCategory.InvalidOperation, this);</a>
</div>
</div>
<div class="rA">Microsoft.PowerShell.Commands.Utility (185)</div><div class="rG" id="Microsoft.PowerShell.Commands.Utility"><div class="rF"><div class="rN">commands\utility\AddMember.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddMember.cs.html#532"><b>532</b>private <i>ErrorRecord</i> NewError(string errorId, string resourceId, object targetObject, params object[] args)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddMember.cs.html#536"><b>536</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\AddType.cs (9)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs.html#233"><b>233</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs.html#251"><b>251</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs.html#342"><b>342</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs.html#584"><b>584</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs.html#747"><b>747</b><i>ErrorRecord</i> errorRecord;</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs.html#1085"><b>1085</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs.html#1096"><b>1096</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs.html#1249"><b>1249</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs.html#1261"><b>1261</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\ConsoleColorCmdlet.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ConsoleColorCmdlet.cs.html#92"><b>92</b>private static <i>ErrorRecord</i> BuildOutOfRangeErrorRecord(object val, string errorId)</a>
</div>
<div class="rF"><div class="rN">commands\utility\ConvertFromMarkdownCommand.cs (4)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs.html#109"><b>109</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs.html#149"><b>149</b><i>ErrorRecord</i> errorRecord = null;</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs.html#207"><b>207</b><i>var</i> errorRecord = new ErrorRecord(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs.html#219"><b>219</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\ConvertTo-Html.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertTo-Html.cs.html#448"><b>448</b><i>ErrorRecord</i> er = new(ex, &quot;ArgumentException&quot;, ErrorCategory.InvalidArgument, &quot;CSSUri&quot;);</a>
</div>
<div class="rF"><div class="rN">commands\utility\CsvCommands.cs (7)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/CsvCommands.cs.html#113"><b>113</b><i>ErrorRecord</i> errorRecord = new(exception, &quot;CannotSpecifyQuoteFieldsAndUseQuotes&quot;, ErrorCategory.InvalidData, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/CsvCommands.cs.html#120"><b>120</b><i>ErrorRecord</i> errorRecord = new(exception, &quot;CannotSpecifyIncludeTypeInformationAndNoTypeInformation&quot;, ErrorCategory.InvalidData, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/CsvCommands.cs.html#263"><b>263</b><i>ErrorRecord</i> errorRecord = new(exception, &quot;CannotSpecifyPathAndLiteralPath&quot;, ErrorCategory.InvalidData, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/CsvCommands.cs.html#468"><b>468</b><i>ErrorRecord</i> errorRecord = new(exception, &quot;CannotAppendCsvWithMismatchedPropertyNames&quot;, ErrorCategory.InvalidData, preexistingPropertyName);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/CsvCommands.cs.html#649"><b>649</b><i>ErrorRecord</i> errorRecord = new(exception, &quot;CannotSpecifyPathAndLiteralPath&quot;, ErrorCategory.InvalidData, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/CsvCommands.cs.html#667"><b>667</b><i>ErrorRecord</i> errorRecord = new(exception, &quot;AlreadyPresentPSMemberInfoInternalCollectionAdd&quot;, ErrorCategory.NotSpecified, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/CsvCommands.cs.html#852"><b>852</b><i>ErrorRecord</i> errorRecord = new(exception, &quot;AlreadyPresentPSMemberInfoInternalCollectionAdd&quot;, ErrorCategory.NotSpecified, null);</a>
</div>
<div class="rF"><div class="rN">commands\utility\EnableDisableRunspaceDebugCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/EnableDisableRunspaceDebugCommand.cs.html#294"><b>294</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\ExportAliasCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ExportAliasCommand.cs.html#412"><b>412</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\FormatAndOutput\format-hex\Format-Hex.cs (3)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs.html#177"><b>177</b><i>ErrorRecord</i> errorRecord = new(e, &quot;FileNotFound&quot;, ErrorCategory.ObjectNotFound, path);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs.html#188"><b>188</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs.html#373"><b>373</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\FormatAndOutput\format-wide\Format-Wide.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-wide/Format-Wide.cs.html#100"><b>100</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\FormatAndOutput\OutGridView\OutGridViewCommand.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/OutGridView/OutGridViewCommand.cs.html#268"><b>268</b><i>ErrorRecord</i> error = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/OutGridView/OutGridViewCommand.cs.html#292"><b>292</b><i>ErrorRecord</i> error = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\GetAliasCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/GetAliasCommand.cs.html#196"><b>196</b><i>ErrorRecord</i> er = new(itemNotFound, &quot;ItemNotFoundException&quot;, ErrorCategory.ObjectNotFound, value);</a>
</div>
<div class="rF"><div class="rN">commands\utility\Get-Error.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs.html#51"><b>51</b>if (InputObject.BaseObject is Exception || InputObject.BaseObject is <i>ErrorRecord</i>)</a>
</div>
<div class="rF"><div class="rN">commands\utility\GetEventCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/GetEventCommand.cs.html#127"><b>127</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\GetEventSubscriberCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/GetEventSubscriberCommand.cs.html#121"><b>121</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\GetHash.cs (4)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs.html#107"><b>107</b><i>ErrorRecord</i> errorRecord = new(e,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs.html#176"><b>176</b><i>var</i> errorRecord = new ErrorRecord(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs.html#185"><b>185</b><i>var</i> errorRecord = new ErrorRecord(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs.html#194"><b>194</b><i>var</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">commands\utility\GetMember.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/GetMember.cs.html#274"><b>274</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\GetRandomCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs.html#98"><b>98</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\Group-Object.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Group-Object.cs.html#405"><b>405</b><i>ErrorRecord</i> er = new(ex, &quot;ArgumentException&quot;, ErrorCategory.InvalidArgument, AsString);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Group-Object.cs.html#412"><b>412</b><i>ErrorRecord</i> er = new(ex, &quot;ArgumentException&quot;, ErrorCategory.InvalidArgument, Property);</a>
</div>
<div class="rF"><div class="rN">commands\utility\ImplicitRemotingCommands.cs (25)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#130"><b>130</b><i>ErrorRecord</i> error = new(new InvalidOperationException(errMsg), &quot;ModuleAndFullyQualifiedModuleCannotBeSpecifiedTogether&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#292"><b>292</b><i>ErrorRecord</i> error = new(new InvalidOperationException(errMsg), &quot;ModuleAndFullyQualifiedModuleCannotBeSpecifiedTogether&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#560"><b>560</b>private <i>ErrorRecord</i> GetErrorNoCommandsImportedBecauseOfSkipping()</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#566"><b>566</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#576"><b>576</b>private <i>ErrorRecord</i> GetErrorMalformedDataFromRemoteCommand(string commandName)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#587"><b>587</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#597"><b>597</b>private <i>ErrorRecord</i> GetErrorCommandSkippedBecauseOfShadowing(string commandNames)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#608"><b>608</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#618"><b>618</b>private <i>ErrorRecord</i> GetErrorSkippedNonRequestedCommand(string commandName)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#629"><b>629</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#639"><b>639</b>private <i>ErrorRecord</i> GetErrorSkippedNonRequestedTypeDefinition(string typeName)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#650"><b>650</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#660"><b>660</b>private <i>ErrorRecord</i> GetErrorSkippedUnsafeCommandName(string commandName)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#671"><b>671</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#681"><b>681</b>private <i>ErrorRecord</i> GetErrorSkippedUnsafeNameInMetadata(string commandName, string nameType, string name)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#703"><b>703</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#713"><b>713</b>private <i>ErrorRecord</i> GetErrorFromRemoteCommand(string commandName, RuntimeException runtimeException)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#727"><b>727</b><i>ErrorRecord</i> errorRecord;</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#767"><b>767</b>private <i>ErrorRecord</i> GetErrorCouldntResolvedAlias(string aliasName)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#778"><b>778</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#788"><b>788</b>private <i>ErrorRecord</i> GetErrorNoResultsFromRemoteEnd(string commandName)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#799"><b>799</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#816"><b>816</b><i>ErrorRecord</i> errorRecord = this.GetErrorCommandSkippedBecauseOfShadowing(skippedCommands);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#1549"><b>1549</b>foreach (<i>ErrorRecord</i> record in powerShell.Streams.Error.ReadAll())</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#1892"><b>1892</b><i>ErrorRecord</i> error = this.GetErrorNoCommandsImportedBecauseOfSkipping();</a>
</div>
<div class="rF"><div class="rN">commands\utility\ImportAliasCommand.cs (3)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImportAliasCommand.cs.html#332"><b>332</b><i>ErrorRecord</i> errorRecord =</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImportAliasCommand.cs.html#354"><b>354</b><i>ErrorRecord</i> errorRecord =</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImportAliasCommand.cs.html#454"><b>454</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\ImportPowerShellDataFile.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImportPowerShellDataFile.cs.html#91"><b>91</b><i>var</i> errorRecord = new ErrorRecord(exception, errorId, errorCategory, path);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImportPowerShellDataFile.cs.html#100"><b>100</b><i>var</i> errorRecord = new ErrorRecord(exception, errorId, errorCategory, resolvedPath);</a>
</div>
<div class="rF"><div class="rN">commands\utility\MarkdownOptionCommands.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/MarkdownOptionCommands.cs.html#158"><b>158</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\MatchString.cs (4)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs.html#1440"><b>1440</b><i>ErrorRecord</i> errorRecord = new(exception, &quot;CannotSpecifyCultureWithoutSimpleMatch&quot;, ErrorCategory.InvalidData, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs.html#1973"><b>1973</b>private static <i>ErrorRecord</i> BuildErrorRecord(string messageId, string argument, string errorId, Exception innerException)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs.html#1978"><b>1978</b>private static <i>ErrorRecord</i> BuildErrorRecord(string messageId, string arg0, string arg1, string errorId, Exception innerException)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs.html#1983"><b>1983</b>private static <i>ErrorRecord</i> BuildErrorRecord(string messageId, object[] arguments, string errorId, Exception innerException)</a>
</div>
<div class="rF"><div class="rN">commands\utility\Measure-Object.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs.html#584"><b>584</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs.html#794"><b>794</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\OrderObjectBase.cs (7)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/OrderObjectBase.cs.html#388"><b>388</b>List&lt;<i>ErrorRecord</i>&gt; evaluationErrors = new();</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/OrderObjectBase.cs.html#392"><b>392</b>foreach (<i>ErrorRecord</i> err in evaluationErrors)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/OrderObjectBase.cs.html#513"><b>513</b>List&lt;<i>ErrorRecord</i>&gt; evaluationErrors = new();</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/OrderObjectBase.cs.html#517"><b>517</b>foreach (<i>ErrorRecord</i> err in evaluationErrors)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/OrderObjectBase.cs.html#545"><b>545</b>List&lt;<i>ErrorRecord</i>&gt; errors, List&lt;string&gt; propertyNotFoundMsgs, bool isCaseSensitive = false, CultureInfo cultureInfo = null, int originalIndex = -1)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/OrderObjectBase.cs.html#579"><b>579</b>List&lt;<i>ErrorRecord</i>&gt; errors,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/OrderObjectBase.cs.html#608"><b>608</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\RegisterPSEventCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/RegisterPSEventCommand.cs.html#45"><b>45</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\RemoveAliasCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/RemoveAliasCommand.cs.html#65"><b>65</b><i>ErrorRecord</i> error = new(notAliasFound, &quot;ItemNotFoundException&quot;, ErrorCategory.ObjectNotFound, aliasName);</a>
</div>
<div class="rF"><div class="rN">commands\utility\RemoveEventCommand.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/RemoveEventCommand.cs.html#114"><b>114</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/RemoveEventCommand.cs.html#127"><b>127</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\Select-Object.cs (7)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs.html#459"><b>459</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs.html#509"><b>509</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs.html#519"><b>519</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs.html#566"><b>566</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs.html#599"><b>599</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs.html#852"><b>852</b>internal <i>ErrorRecord</i> ErrorRecord { get; }</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs.html#854"><b>854</b>internal SelectObjectException(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">commands\utility\Send-MailMessage.cs (8)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs.html#219"><b>219</b><i>ErrorRecord</i> er = new(e, &quot;FormatException&quot;, ErrorCategory.InvalidType, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs.html#242"><b>242</b><i>ErrorRecord</i> er = new(e, &quot;FormatException&quot;, ErrorCategory.InvalidType, From);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs.html#297"><b>297</b><i>ErrorRecord</i> er = new(new InvalidOperationException(SendMailMessageStrings.HostNameValue), null, ErrorCategory.InvalidArgument, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs.html#365"><b>365</b><i>ErrorRecord</i> er = new(ex, &quot;SmtpFailedRecipientsException&quot;, ErrorCategory.InvalidOperation, _mSmtpClient);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs.html#372"><b>372</b><i>ErrorRecord</i> er = new(new SmtpException(ex.InnerException.Message), &quot;SmtpException&quot;, ErrorCategory.InvalidOperation, _mSmtpClient);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs.html#377"><b>377</b><i>ErrorRecord</i> er = new(ex, &quot;SmtpException&quot;, ErrorCategory.InvalidOperation, _mSmtpClient);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs.html#383"><b>383</b><i>ErrorRecord</i> er = new(ex, &quot;InvalidOperationException&quot;, ErrorCategory.InvalidOperation, _mSmtpClient);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs.html#388"><b>388</b><i>ErrorRecord</i> er = new(ex, &quot;AuthenticationException&quot;, ErrorCategory.InvalidOperation, _mSmtpClient);</a>
</div>
<div class="rF"><div class="rN">commands\utility\ShowCommand\ShowCommand.cs (5)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs.html#53"><b>53</b>private PSDataCollection&lt;<i>ErrorRecord</i>&gt; _errors = new();</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs.html#177"><b>177</b><i>ErrorRecord</i> error = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs.html#187"><b>187</b><i>ErrorRecord</i> error = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs.html#245"><b>245</b><i>ErrorRecord</i> error = _errors[i];</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs.html#466"><b>466</b>this.WriteError(((PSDataCollection&lt;<i>ErrorRecord</i>&gt;)sender)[e.Index]);</a>
</div>
<div class="rF"><div class="rN">commands\utility\ShowMarkdownCommand.cs (5)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs.html#118"><b>118</b>foreach (<i>ErrorRecord</i> errorRecord in _powerShell.Streams.Error)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs.html#153"><b>153</b><i>var</i> errorRecord = new ErrorRecord(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs.html#178"><b>178</b><i>var</i> errorRecord = new ErrorRecord(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs.html#191"><b>191</b><i>var</i> errorRecord = new ErrorRecord(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs.html#211"><b>211</b><i>var</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">commands\utility\TestJsonCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs.html#156"><b>156</b><i>ErrorRecord</i> errorRecord = new(exception, &quot;InvalidJsonAgainstSchema&quot;, ErrorCategory.InvalidData, null);</a>
</div>
<div class="rF"><div class="rN">commands\utility\trace\TraceCommandBase.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceCommandBase.cs.html#110"><b>110</b><i>ErrorRecord</i> errorRecord = new(itemNotFound.ErrorRecord, itemNotFound);</a>
</div>
<div class="rF"><div class="rN">commands\utility\trace\TraceExpressionCommand.cs (7)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceExpressionCommand.cs.html#451"><b>451</b><i>ErrorRecord</i> errorRecord = ConvertToErrorRecord(obj);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceExpressionCommand.cs.html#496"><b>496</b><i>ErrorRecord</i> errorRecord = ConvertToErrorRecord(o);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceExpressionCommand.cs.html#506"><b>506</b><i>ErrorRecord</i> errorRecord = ConvertToErrorRecord(obj);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceExpressionCommand.cs.html#525"><b>525</b>private static <i>ErrorRecord</i> ConvertToErrorRecord(object obj)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceExpressionCommand.cs.html#527"><b>527</b><i>ErrorRecord</i> result = null;</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceExpressionCommand.cs.html#538"><b>538</b><i>ErrorRecord</i> errorRecordResult = obj as <i>ErrorRecord</i>;</a>
</div>
<div class="rF"><div class="rN">commands\utility\trace\TraceListenerCommandBase.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceListenerCommandBase.cs.html#336"><b>336</b><i>ErrorRecord</i> errorRecord =</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceListenerCommandBase.cs.html#361"><b>361</b><i>ErrorRecord</i> errorRecord =</a>
</div>
<div class="rF"><div class="rN">commands\utility\UnblockFile.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs.html#113"><b>113</b><i>ErrorRecord</i> errorRecord = new(e,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs.html#183"><b>183</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\UnregisterEventCommand.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/UnregisterEventCommand.cs.html#100"><b>100</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/UnregisterEventCommand.cs.html#114"><b>114</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\Update-Data.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-Data.cs.html#38"><b>38</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-Data.cs.html#48"><b>48</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\Update-List.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-List.cs.html#181"><b>181</b>private <i>ErrorRecord</i> NewError(string errorId, string resourceId, object targetObject, params object[] args)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-List.cs.html#184"><b>184</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\Update-TypeData.cs (4)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#752"><b>752</b>private static <i>ErrorRecord</i> NewError(string errorId, string template, object targetObject, params object[] args)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#755"><b>755</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#1261"><b>1261</b>private static <i>ErrorRecord</i> NewError(string errorId, string template, object targetObject, params object[] args)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#1264"><b>1264</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">commands\utility\WebCmdlet\Common\InvokeRestMethodCommand.Common.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs.html#189"><b>189</b><i>ErrorRecord</i> error;</a>
</div>
<div class="rF"><div class="rN">commands\utility\WebCmdlet\Common\WebRequestPSCmdlet.Common.cs (28)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#403"><b>403</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.SessionConflict,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#411"><b>411</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.AuthenticationConflict,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#418"><b>418</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.AuthenticationTokenConflict,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#425"><b>425</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.AuthenticationCredentialNotSupplied,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#432"><b>432</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.AuthenticationTokenNotSupplied,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#439"><b>439</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.AllowUnencryptedAuthenticationRequired,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#446"><b>446</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.AllowUnencryptedAuthenticationRequired,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#454"><b>454</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.CredentialConflict,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#462"><b>462</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.ProxyCredentialConflict,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#468"><b>468</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.ProxyUriNotSupplied,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#476"><b>476</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.BodyConflict,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#483"><b>483</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.BodyFormConflict,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#490"><b>490</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.FormInFileConflict,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#499"><b>499</b><i>ErrorRecord</i> errorRecord = null;</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#557"><b>557</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.OutFileMissing,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#565"><b>565</b><i>ErrorRecord</i> error = GetValidationError(WebCmdletStrings.OutFileMissing,</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#801"><b>801</b>private <i>ErrorRecord</i> GetValidationError(string msg, string errorId)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#804"><b>804</b><i>var</i> error = new ErrorRecord(ex, errorId, ErrorCategory.InvalidArgument, this);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#808"><b>808</b>private <i>ErrorRecord</i> GetValidationError(string msg, string errorId, params object[] args)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#812"><b>812</b><i>var</i> error = new ErrorRecord(ex, errorId, ErrorCategory.InvalidArgument, this);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#1287"><b>1287</b><i>ErrorRecord</i> er = new(outerEx, &quot;WebCmdletContentTypeException&quot;, ErrorCategory.InvalidArgument, ContentType);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#1543"><b>1543</b><i>ErrorRecord</i> er = new(httpEx, &quot;WebCmdletWebResponseException&quot;, ErrorCategory.InvalidOperation, request);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#1590"><b>1590</b><i>ErrorRecord</i> er = new(new InvalidOperationException(), &quot;MaximumRedirectExceeded&quot;, ErrorCategory.InvalidOperation, request);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#1598"><b>1598</b><i>ErrorRecord</i> er = new(ex, &quot;WebCmdletWebResponseException&quot;, ErrorCategory.InvalidOperation, request);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#1624"><b>1624</b><i>ErrorRecord</i> er = new(ex, &quot;WebCmdletCertificateException&quot;, ErrorCategory.SecurityError, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#1629"><b>1629</b><i>ErrorRecord</i> er = new(ex, &quot;WebCmdletIEDomNotSupportedException&quot;, ErrorCategory.NotImplemented, null);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#1709"><b>1709</b><i>ErrorRecord</i> er = new(outerEx, &quot;WebCmdletContentTypeException&quot;, ErrorCategory.InvalidArgument, ContentType);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs.html#1718"><b>1718</b><i>ErrorRecord</i> er = new(outerEx, &quot;WebCmdletContentTypeException&quot;, ErrorCategory.InvalidArgument, ContentType);</a>
</div>
<div class="rF"><div class="rN">commands\utility\WebCmdlet\ConvertFromJsonCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertFromJsonCommand.cs.html#115"><b>115</b><i>ErrorRecord</i> error = null;</a>
</div>
<div class="rF"><div class="rN">commands\utility\WebCmdlet\JsonObject.cs (8)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs.html#120"><b>120</b>public static object ConvertFromJson(string input, out <i>ErrorRecord</i> error)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs.html#136"><b>136</b>public static object ConvertFromJson(string input, bool returnHashtable, out <i>ErrorRecord</i> error)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs.html#153"><b>153</b>public static object ConvertFromJson(string input, bool returnHashtable, int? maxDepth, out <i>ErrorRecord</i> error)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs.html#214"><b>214</b>private static PSObject PopulateFromJDictionary(JObject entries, DuplicateMemberHashSet memberHashTracker, out <i>ErrorRecord</i> error)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs.html#298"><b>298</b>private static ICollection&lt;object&gt; PopulateFromJArray(JArray list, out <i>ErrorRecord</i> error)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs.html#344"><b>344</b>private static Hashtable PopulateHashTableFromJDictionary(JObject entries, out <i>ErrorRecord</i> error)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs.html#401"><b>401</b>private static ICollection&lt;object&gt; PopulateHashTableFromJArray(JArray list, out <i>ErrorRecord</i> error)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs.html#733"><b>733</b><i>var</i> errorRecord = new ErrorRecord(exception, &quot;NonStringKeyInDictionary&quot;, ErrorCategory.InvalidOperation, dict);</a>
</div>
<div class="rF"><div class="rN">commands\utility\Write.cs (3)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Write.cs.html#187"><b>187</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Write.cs.html#236"><b>236</b>public <i>ErrorRecord</i> ErrorRecord { get; set; }</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Write.cs.html#305"><b>305</b><i>ErrorRecord</i> errorRecord = this.ErrorRecord;</a>
</div>
<div class="rF"><div class="rN">commands\utility\XmlCommands.cs (4)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs.html#975"><b>975</b><i>ErrorRecord</i> errorRecord = new(argumentException, &quot;ProcessingFile&quot;, ErrorCategory.InvalidArgument, filePath);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs.html#1003"><b>1003</b><i>ErrorRecord</i> er = new(e, &quot;PrefixError&quot;, ErrorCategory.InvalidOperation, namespacetable);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs.html#1010"><b>1010</b><i>ErrorRecord</i> er = new(e, &quot;PrefixError&quot;, ErrorCategory.InvalidOperation, namespacetable);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs.html#1056"><b>1056</b><i>ErrorRecord</i> er = new(e, &quot;ProcessingFile&quot;, ErrorCategory.InvalidOperation, fpath);</a>
</div>
</div>
<div class="rA">Microsoft.PowerShell.ConsoleHost (6)</div><div class="rG" id="Microsoft.PowerShell.ConsoleHost"><div class="rF"><div class="rN">host\msh\ConsoleHost.cs (1)</div>
<a href="../../Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs.html#2135"><b>2135</b><i>ErrorRecord</i> er = null;</a>
</div>
<div class="rF"><div class="rN">host\msh\Executor.cs (1)</div>
<a href="../../Microsoft.PowerShell.ConsoleHost/host/msh/Executor.cs.html#109"><b>109</b><i>ErrorRecord</i> er = null;</a>
</div>
<div class="rF"><div class="rN">host\msh\StartTranscriptCmdlet.cs (4)</div>
<a href="../../Microsoft.PowerShell.ConsoleHost/host/msh/StartTranscriptCmdlet.cs.html#200"><b>200</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../../Microsoft.PowerShell.ConsoleHost/host/msh/StartTranscriptCmdlet.cs.html#263"><b>263</b><i>ErrorRecord</i> er = new ErrorRecord(</a>
<a href="../../Microsoft.PowerShell.ConsoleHost/host/msh/StartTranscriptCmdlet.cs.html#326"><b>326</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../../Microsoft.PowerShell.ConsoleHost/host/msh/StartTranscriptCmdlet.cs.html#336"><b>336</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
</div>
<div class="rA">Microsoft.PowerShell.Security (44)</div><div class="rG" id="Microsoft.PowerShell.Security"><div class="rF"><div class="rN">security\AclCommands.cs (9)</div>
<a href="../../Microsoft.PowerShell.Security/security/AclCommands.cs.html#776"><b>776</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/AclCommands.cs.html#790"><b>790</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/AclCommands.cs.html#831"><b>831</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/AclCommands.cs.html#856"><b>856</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/AclCommands.cs.html#867"><b>867</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/AclCommands.cs.html#1348"><b>1348</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/AclCommands.cs.html#1360"><b>1360</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/AclCommands.cs.html#1410"><b>1410</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/AclCommands.cs.html#1545"><b>1545</b><i>ErrorRecord</i> er =</a>
</div>
<div class="rF"><div class="rN">security\CertificateCommands.cs (3)</div>
<a href="../../Microsoft.PowerShell.Security/security/CertificateCommands.cs.html#163"><b>163</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/CertificateCommands.cs.html#181"><b>181</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/CertificateCommands.cs.html#196"><b>196</b><i>ErrorRecord</i> er =</a>
</div>
<div class="rF"><div class="rN">security\CertificateProvider.cs (4)</div>
<a href="../../Microsoft.PowerShell.Security/security/CertificateProvider.cs.html#1446"><b>1446</b>private static <i>ErrorRecord</i> CreateErrorRecord(string path,</a>
<a href="../../Microsoft.PowerShell.Security/security/CertificateProvider.cs.html#1502"><b>1502</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/CertificateProvider.cs.html#1538"><b>1538</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/CertificateProvider.cs.html#1552"><b>1552</b><i>ErrorRecord</i> er = CreateErrorRecord(path, itemType);</a>
</div>
<div class="rF"><div class="rN">security\CmsCommands.cs (7)</div>
<a href="../../Microsoft.PowerShell.Security/security/CmsCommands.cs.html#97"><b>97</b><i>ErrorRecord</i> error = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/CmsCommands.cs.html#172"><b>172</b><i>ErrorRecord</i> terminatingError = null;</a>
<a href="../../Microsoft.PowerShell.Security/security/CmsCommands.cs.html#252"><b>252</b><i>ErrorRecord</i> error = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/CmsCommands.cs.html#315"><b>315</b><i>ErrorRecord</i> error = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/CmsCommands.cs.html#433"><b>433</b><i>ErrorRecord</i> error = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/CmsCommands.cs.html#527"><b>527</b><i>ErrorRecord</i> error = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/CmsCommands.cs.html#560"><b>560</b><i>ErrorRecord</i> error = null;</a>
</div>
<div class="rF"><div class="rN">security\CredentialCommands.cs (1)</div>
<a href="../../Microsoft.PowerShell.Security/security/CredentialCommands.cs.html#107"><b>107</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">security\ExecutionPolicyCommands.cs (4)</div>
<a href="../../Microsoft.PowerShell.Security/security/ExecutionPolicyCommands.cs.html#68"><b>68</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/ExecutionPolicyCommands.cs.html#178"><b>178</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/ExecutionPolicyCommands.cs.html#223"><b>223</b><i>ErrorRecord</i> errorRecord = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/ExecutionPolicyCommands.cs.html#331"><b>331</b><i>ErrorRecord</i> errorRecord = new(</a>
</div>
<div class="rF"><div class="rN">security\SecureStringCommands.cs (2)</div>
<a href="../../Microsoft.PowerShell.Security/security/SecureStringCommands.cs.html#367"><b>367</b><i>ErrorRecord</i> er =</a>
<a href="../../Microsoft.PowerShell.Security/security/SecureStringCommands.cs.html#376"><b>376</b><i>ErrorRecord</i> er =</a>
</div>
<div class="rF"><div class="rN">security\SignatureCommands.cs (6)</div>
<a href="../../Microsoft.PowerShell.Security/security/SignatureCommands.cs.html#486"><b>486</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/SignatureCommands.cs.html#496"><b>496</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/SignatureCommands.cs.html#506"><b>506</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/SignatureCommands.cs.html#516"><b>516</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/SignatureCommands.cs.html#526"><b>526</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/SignatureCommands.cs.html#550"><b>550</b><i>ErrorRecord</i> er = SecurityUtils.CreateInvalidArgumentErrorRecord(</a>
</div>
<div class="rF"><div class="rN">security\Utils.cs (8)</div>
<a href="../../Microsoft.PowerShell.Security/security/Utils.cs.html#57"><b>57</b><i>ErrorRecord</i> CreateFileNotFoundErrorRecord(string resourceStr,</a>
<a href="../../Microsoft.PowerShell.Security/security/Utils.cs.html#69"><b>69</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/Utils.cs.html#84"><b>84</b><i>ErrorRecord</i> CreatePathNotFoundErrorRecord(string path,</a>
<a href="../../Microsoft.PowerShell.Security/security/Utils.cs.html#89"><b>89</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/Utils.cs.html#106"><b>106</b><i>ErrorRecord</i> CreateNotSupportedErrorRecord(string resourceStr,</a>
<a href="../../Microsoft.PowerShell.Security/security/Utils.cs.html#114"><b>114</b><i>ErrorRecord</i> er = new(</a>
<a href="../../Microsoft.PowerShell.Security/security/Utils.cs.html#130"><b>130</b><i>ErrorRecord</i> CreateInvalidArgumentErrorRecord(Exception e,</a>
<a href="../../Microsoft.PowerShell.Security/security/Utils.cs.html#133"><b>133</b><i>ErrorRecord</i> er = new(</a>
</div>
</div>
<div class="rA">Microsoft.WSMan.Management (47)</div><div class="rG" id="Microsoft.WSMan.Management"><div class="rF"><div class="rN">ConfigProvider.cs (12)</div>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2118"><b>2118</b><i>ErrorRecord</i> er = new ErrorRecord(new InvalidOperationException(helper.GetResourceMsgFromResourcetext(&quot;NewItemNotSupported&quot;)), &quot;InvalidOperationException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2399"><b>2399</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;COMException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2464"><b>2464</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;ArgumentNullException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2469"><b>2469</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;UnauthorizedAccessException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2474"><b>2474</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;NotSupportedException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2479"><b>2479</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;FileNotFoundException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2484"><b>2484</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;DirectoryNotFoundException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2489"><b>2489</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;SecurityException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2519"><b>2519</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;ArgumentNullException&quot;, ErrorCategory.InvalidArgument, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#2632"><b>2632</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;XPathException&quot;, ErrorCategory.InvalidArgument, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#4573"><b>4573</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;WsManError&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/ConfigProvider.cs.html#4580"><b>4580</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;WsManError&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">CredSSP.cs (22)</div>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#80"><b>80</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;COMException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#131"><b>131</b><i>ErrorRecord</i> er = new ErrorRecord(ex, helper.GetResourceMsgFromResourcetext(&quot;WinrmNotConfigured&quot;), ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#160"><b>160</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;XpathException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#196"><b>196</b><i>ErrorRecord</i> er = new ErrorRecord(ex,</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#311"><b>311</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;InvalidOperation&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#316"><b>316</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;InvalidArgument&quot;, ErrorCategory.InvalidArgument, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#321"><b>321</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;SecurityException&quot;, ErrorCategory.SecurityError, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#326"><b>326</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;UnauthorizedAccess&quot;, ErrorCategory.SecurityError, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#506"><b>506</b><i>ErrorRecord</i> er = new ErrorRecord(ex, helper.GetResourceMsgFromResourcetext(&quot;WinrmNotConfigured&quot;), ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#586"><b>586</b><i>ErrorRecord</i> er = new ErrorRecord(ex, helper.GetResourceMsgFromResourcetext(&quot;WinrmNotConfigured&quot;), ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#680"><b>680</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;UnauthorizedAccessException&quot;, ErrorCategory.PermissionDenied, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#685"><b>685</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;SecurityException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#690"><b>690</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;ArgumentException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#787"><b>787</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;ArgumentException&quot;, ErrorCategory.PermissionDenied, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#792"><b>792</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;SecurityException&quot;, ErrorCategory.PermissionDenied, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#797"><b>797</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;ObjectDisposedException&quot;, ErrorCategory.PermissionDenied, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#824"><b>824</b><i>ErrorRecord</i> er = new ErrorRecord(ex, helper.GetResourceMsgFromResourcetext(&quot;WinrmNotConfigured&quot;), ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#847"><b>847</b><i>ErrorRecord</i> er = new ErrorRecord(ex, helper.GetResourceMsgFromResourcetext(&quot;WinrmNotConfigured&quot;), ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#863"><b>863</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;UnauthorizedAccess&quot;, ErrorCategory.PermissionDenied, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#868"><b>868</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;SecurityException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#873"><b>873</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;InvalidArgument&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/CredSSP.cs.html#878"><b>878</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;XPathException&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">NewWSManSession.cs (2)</div>
<a href="../../Microsoft.WSMan.Management/NewWSManSession.cs.html#267"><b>267</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;InvalidOperationException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/NewWSManSession.cs.html#276"><b>276</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;InvalidOperationException&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">PingWSMan.cs (1)</div>
<a href="../../Microsoft.WSMan.Management/PingWSMan.cs.html#167"><b>167</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;WsManError&quot;, ErrorCategory.InvalidOperation, computername);</a>
</div>
<div class="rF"><div class="rN">Set-QuickConfig.cs (2)</div>
<a href="../../Microsoft.WSMan.Management/Set-QuickConfig.cs.html#188"><b>188</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;InvalidOperation&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../../Microsoft.WSMan.Management/Set-QuickConfig.cs.html#207"><b>207</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;InvalidOperation&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">WsManHelper.cs (7)</div>
<a href="../../Microsoft.WSMan.Management/WsManHelper.cs.html#348"><b>348</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;ArgumentNullException&quot;, ErrorCategory.InvalidArgument, null);</a>
<a href="../../Microsoft.WSMan.Management/WsManHelper.cs.html#353"><b>353</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;UnauthorizedAccessException&quot;, ErrorCategory.PermissionDenied, null);</a>
<a href="../../Microsoft.WSMan.Management/WsManHelper.cs.html#358"><b>358</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;FileNotFoundException&quot;, ErrorCategory.ObjectNotFound, null);</a>
<a href="../../Microsoft.WSMan.Management/WsManHelper.cs.html#363"><b>363</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;DirectoryNotFoundException&quot;, ErrorCategory.ObjectNotFound, null);</a>
<a href="../../Microsoft.WSMan.Management/WsManHelper.cs.html#368"><b>368</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;SecurityException&quot;, ErrorCategory.SecurityError, null);</a>
<a href="../../Microsoft.WSMan.Management/WsManHelper.cs.html#891"><b>891</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;WsManError&quot;, ErrorCategory.InvalidOperation, targetobject);</a>
<a href="../../Microsoft.WSMan.Management/WsManHelper.cs.html#904"><b>904</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;WsManError&quot;, ErrorCategory.InvalidOperation, targetobject);</a>
</div>
<div class="rF"><div class="rN">WSManInstance.cs (1)</div>
<a href="../../Microsoft.WSMan.Management/WSManInstance.cs.html#556"><b>556</b><i>ErrorRecord</i> er = new ErrorRecord(ex, &quot;Exception&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
</div>
<div class="rA">System.Management.Automation (564)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">DscSupport\CimDSCParser.cs (13)</div>
<a href="../DscSupport/CimDSCParser.cs.html#3389"><b>3389</b>public static <i>ErrorRecord</i> GetBadlyFormedRequiredResourceIdErrorRecord(string badDependsOnReference, string definingResource)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3404"><b>3404</b>public static <i>ErrorRecord</i> GetBadlyFormedExclusiveResourceIdErrorRecord(string badExclusiveResourcereference, string definingResource)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3418"><b>3418</b>public static <i>ErrorRecord</i> GetPullModeNeedConfigurationSource(string resourceId)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3432"><b>3432</b>public static <i>ErrorRecord</i> DisabledRefreshModeNotValidForPartialConfig(string resourceId)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3447"><b>3447</b>public static <i>ErrorRecord</i> DuplicateResourceIdInNodeStatementErrorRecord(string duplicateResourceId, string nodeName)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3461"><b>3461</b>public static <i>ErrorRecord</i> InvalidConfigurationNameErrorRecord(string configurationName)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3478"><b>3478</b>public static <i>ErrorRecord</i> InvalidValueForPropertyErrorRecord(string propertyName, string value, string keywordName, string validValues)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3493"><b>3493</b>public static <i>ErrorRecord</i> InvalidLocalConfigurationManagerPropertyErrorRecord(string propertyName, string validProperties)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3510"><b>3510</b>public static <i>ErrorRecord</i> UnsupportedValueForPropertyErrorRecord(string propertyName, string value, string keywordName, string validValues)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3526"><b>3526</b>public static <i>ErrorRecord</i> MissingValueForMandatoryPropertyErrorRecord(string keywordName, string typeName, string propertyName)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3539"><b>3539</b>public static <i>ErrorRecord</i> DebugModeShouldHaveOneValue()</a>
<a href="../DscSupport/CimDSCParser.cs.html#3557"><b>3557</b>public static <i>ErrorRecord</i> ValueNotInRangeErrorRecord(string property, string name, int providedValue, int lower, int upper)</a>
<a href="../DscSupport/CimDSCParser.cs.html#3571"><b>3571</b>public static <i>ErrorRecord</i> PsDscRunAsCredentialMergeErrorForCompositeResources(string resourceId)</a>
</div>
<div class="rF"><div class="rN">engine\cmdlet.cs (5)</div>
<a href="../engine/cmdlet.cs.html#341"><b>341</b>public void WriteError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/cmdlet.cs.html#371"><b>371</b>/// &lt;seealso cref=&quot;System.Management.Automation.ICommandRuntime.WriteError(<i>ErrorRecord</i>)&quot;/&gt;</a>
<a href="../engine/cmdlet.cs.html#407"><b>407</b>/// &lt;seealso cref=&quot;System.Management.Automation.ICommandRuntime.WriteError(<i>ErrorRecord</i>)&quot;/&gt;</a>
<a href="../engine/cmdlet.cs.html#1705"><b>1705</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;</a>
<a href="../engine/cmdlet.cs.html#1718"><b>1718</b>public void ThrowTerminatingError(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">engine\CommandDiscovery.cs (1)</div>
<a href="../engine/CommandDiscovery.cs.html#296"><b>296</b><i>ErrorRecord</i> error = null;</a>
</div>
<div class="rF"><div class="rN">engine\CommandProcessor.cs (2)</div>
<a href="../engine/CommandProcessor.cs.html#547"><b>547</b><i>ErrorRecord</i> errorRecord =</a>
<a href="../engine/CommandProcessor.cs.html#623"><b>623</b><i>ErrorRecord</i> errorRecord =</a>
</div>
<div class="rF"><div class="rN">engine\CommandProcessorBase.cs (3)</div>
<a href="../engine/CommandProcessorBase.cs.html#50"><b>50</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/CommandProcessorBase.cs.html#222"><b>222</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/CommandProcessorBase.cs.html#845"><b>845</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">engine\DefaultCommandRuntime.cs (2)</div>
<a href="../engine/DefaultCommandRuntime.cs.html#48"><b>48</b>public void WriteError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/DefaultCommandRuntime.cs.html#234"><b>234</b>public void ThrowTerminatingError(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">engine\ErrorPackage.cs (28)</div>
<a href="../engine/ErrorPackage.cs.html#25"><b>25</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;.</a>
<a href="../engine/ErrorPackage.cs.html#35"><b>35</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;.</a>
<a href="../engine/ErrorPackage.cs.html#189"><b>189</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;</a>
<a href="../engine/ErrorPackage.cs.html#194"><b>194</b>internal ErrorCategoryInfo(<i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/ErrorPackage.cs.html#317"><b>317</b>return <i>ErrorRecord</i>.NotNull(targetInString);</a>
<a href="../engine/ErrorPackage.cs.html#480"><b>480</b>private readonly <i>ErrorRecord</i> _errorRecord;</a>
<a href="../engine/ErrorPackage.cs.html#516"><b>516</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;</a>
<a href="../engine/ErrorPackage.cs.html#520"><b>520</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;,</a>
<a href="../engine/ErrorPackage.cs.html#738"><b>738</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.Exception&quot;/&gt;</a>
<a href="../engine/ErrorPackage.cs.html#742"><b>742</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;</a>
<a href="../engine/ErrorPackage.cs.html#744"><b>744</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.ErrorDetails&quot;/&gt;</a>
<a href="../engine/ErrorPackage.cs.html#749"><b>749</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.Exception&quot;/&gt;.</a>
<a href="../engine/ErrorPackage.cs.html#756"><b>756</b>get { return <i>ErrorRecord</i>.NotNull(_message); }</a>
<a href="../engine/ErrorPackage.cs.html#774"><b>774</b>return <i>ErrorRecord</i>.NotNull(_recommendedAction);</a>
<a href="../engine/ErrorPackage.cs.html#970"><b>970</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.ErrorDetails&quot;/&gt;,</a>
<a href="../engine/ErrorPackage.cs.html#971"><b>971</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.TargetObject&quot;/&gt;,</a>
<a href="../engine/ErrorPackage.cs.html#972"><b>972</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.CategoryInfo&quot;/&gt;,</a>
<a href="../engine/ErrorPackage.cs.html#973"><b>973</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.FullyQualifiedErrorId&quot;/&gt;,</a>
<a href="../engine/ErrorPackage.cs.html#974"><b>974</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.ErrorDetails&quot;/&gt;, and</a>
<a href="../engine/ErrorPackage.cs.html#975"><b>975</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.InvocationInfo&quot;/&gt;.</a>
<a href="../engine/ErrorPackage.cs.html#977"><b>977</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;</a>
<a href="../engine/ErrorPackage.cs.html#1282"><b>1282</b>internal static <i>ErrorRecord</i> FromPSObjectForRemoting(PSObject serializedErrorRecord)</a>
<a href="../engine/ErrorPackage.cs.html#1284"><b>1284</b><i>ErrorRecord</i> er = new ErrorRecord();</a>
<a href="../engine/ErrorPackage.cs.html#1395"><b>1395</b>public ErrorRecord(<i>ErrorRecord</i> errorRecord,</a>
<a href="../engine/ErrorPackage.cs.html#1443"><b>1443</b>internal virtual <i>ErrorRecord</i> WrapException(Exception replaceParentContainsErrorRecordException)</a>
<a href="../engine/ErrorPackage.cs.html#1725"><b>1725</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;</a>
<a href="../engine/ErrorPackage.cs.html#1779"><b>1779</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.Exception&quot;/&gt;</a>
<a href="../engine/ErrorPackage.cs.html#1794"><b>1794</b><i>ErrorRecord</i> ErrorRecord { get; }</a>
</div>
<div class="rF"><div class="rN">engine\EventManager.cs (2)</div>
<a href="../engine/EventManager.cs.html#2650"><b>2650</b><i>var</i> error = (<i>ErrorRecord</i>)errors[i];</a>
</div>
<div class="rF"><div class="rN">engine\ExecutionContext.cs (6)</div>
<a href="../engine/ExecutionContext.cs.html#842"><b>842</b><i>ErrorRecord</i> objAsErrorRecord = obj as <i>ErrorRecord</i>;</a>
<a href="../engine/ExecutionContext.cs.html#865"><b>865</b><i>ErrorRecord</i> er1 = arraylist[0] as <i>ErrorRecord</i>;</a>
<a href="../engine/ExecutionContext.cs.html#1467"><b>1467</b><i>ErrorRecord</i> error = null;</a>
<a href="../engine/ExecutionContext.cs.html#1494"><b>1494</b>internal void ReportEngineStartupError(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">engine\ExtendedTypeSystemException.cs (2)</div>
<a href="../engine/ExtendedTypeSystemException.cs.html#646"><b>646</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../engine/ExtendedTypeSystemException.cs.html#663"><b>663</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
<div class="rF"><div class="rN">engine\GetCommandCommand.cs (2)</div>
<a href="../engine/GetCommandCommand.cs.html#398"><b>398</b><i>ErrorRecord</i> error = new ErrorRecord(new InvalidOperationException(errMsg), &quot;ModuleAndFullyQualifiedModuleCannotBeSpecifiedTogether&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../engine/GetCommandCommand.cs.html#475"><b>475</b><i>var</i> errorRecord = new ErrorRecord(exception, &quot;CommandParameterNotFound&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\History.cs (3)</div>
<a href="../engine/hostifaces/History.cs.html#1103"><b>1103</b>EventHandler&lt;DataAddedEventArgs&gt; errorAdded = (object sender, DataAddedEventArgs e) =&gt; { <i>ErrorRecord</i> record = (<i>ErrorRecord</i>)((PSDataCollection&lt;<i>ErrorRecord</i>&gt;)sender)[e.Index]; WriteError(record); };</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\HostUtilities.cs (3)</div>
<a href="../engine/hostifaces/HostUtilities.cs.html#333"><b>333</b><i>ErrorRecord</i> lastErrorRecord = null;</a>
<a href="../engine/hostifaces/HostUtilities.cs.html#338"><b>338</b>lastErrorRecord = errorList[0] as <i>ErrorRecord</i>;</a>
<a href="../engine/hostifaces/HostUtilities.cs.html#479"><b>479</b>if (lastError != null &amp;&amp; lastError is <i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\InternalHostUserInterface.cs (1)</div>
<a href="../engine/hostifaces/InternalHostUserInterface.cs.html#368"><b>368</b><i>ErrorRecord</i> errorRecord = null;</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\LocalConnection.cs (3)</div>
<a href="../engine/hostifaces/LocalConnection.cs.html#1543"><b>1543</b>PSDataCollection&lt;<i>ErrorRecord</i>&gt; errors)</a>
<a href="../engine/hostifaces/LocalConnection.cs.html#1557"><b>1557</b>public PSDataCollection&lt;<i>ErrorRecord</i>&gt; ErrorRecords</a>
<a href="../engine/hostifaces/LocalConnection.cs.html#1562"><b>1562</b>private readonly PSDataCollection&lt;<i>ErrorRecord</i>&gt; _errors;</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\PowerShell.cs (9)</div>
<a href="../engine/hostifaces/PowerShell.cs.html#589"><b>589</b>private PSDataCollection&lt;<i>ErrorRecord</i>&gt; _errorBuffer;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#637"><b>637</b>_errorBuffer = new PSDataCollection&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../engine/hostifaces/PowerShell.cs.html#699"><b>699</b>PSDataCollectionStream&lt;<i>ErrorRecord</i>&gt; errordatastream = (PSDataCollectionStream&lt;<i>ErrorRecord</i>&gt;)errorstream;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#1518"><b>1518</b>internal PSDataCollection&lt;<i>ErrorRecord</i>&gt; ErrorBuffer</a>
<a href="../engine/hostifaces/PowerShell.cs.html#2057"><b>2057</b>new PSDataCollectionStream&lt;<i>ErrorRecord</i>&gt;(InstanceId, _errorBuffer),</a>
<a href="../engine/hostifaces/PowerShell.cs.html#4890"><b>4890</b>new PSDataCollectionStream&lt;<i>ErrorRecord</i>&gt;(InstanceId, _errorBuffer),</a>
<a href="../engine/hostifaces/PowerShell.cs.html#5399"><b>5399</b>_errorStream = new PSDataCollectionStream&lt;<i>ErrorRecord</i>&gt;(shell.InstanceId, shell._errorBuffer);</a>
<a href="../engine/hostifaces/PowerShell.cs.html#6032"><b>6032</b>public PSDataCollection&lt;<i>ErrorRecord</i>&gt; Error</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\PSTask.cs (1)</div>
<a href="../engine/hostifaces/PSTask.cs.html#157"><b>157</b><i>var</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">engine\ICommandRuntime.cs (3)</div>
<a href="../engine/ICommandRuntime.cs.html#57"><b>57</b>void WriteError(<i>ErrorRecord</i> errorRecord);</a>
<a href="../engine/ICommandRuntime.cs.html#547"><b>547</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;</a>
<a href="../engine/ICommandRuntime.cs.html#555"><b>555</b>void ThrowTerminatingError(<i>ErrorRecord</i> errorRecord);</a>
</div>
<div class="rF"><div class="rN">engine\InitialSessionState.cs (5)</div>
<a href="../engine/InitialSessionState.cs.html#2918"><b>2918</b><i>ErrorRecord</i> lastErrorRecord = errorList[0] as <i>ErrorRecord</i>;</a>
<a href="../engine/InitialSessionState.cs.html#3223"><b>3223</b><i>ErrorRecord</i> er;</a>
<a href="../engine/InitialSessionState.cs.html#3226"><b>3226</b>PSDataCollection&lt;<i>ErrorRecord</i>&gt; mergedErrors = new PSDataCollection&lt;<i>ErrorRecord</i>&gt;();</a>
</div>
<div class="rF"><div class="rN">engine\InternalCommands.cs (6)</div>
<a href="../engine/InternalCommands.cs.html#687"><b>687</b><i>ErrorRecord</i> errorRecord = null;</a>
<a href="../engine/InternalCommands.cs.html#976"><b>976</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/InternalCommands.cs.html#1243"><b>1243</b>internal static <i>ErrorRecord</i> GenerateNameParameterError(string paraName, string resourceString, string errorId, object target, params object[] args)</a>
<a href="../engine/InternalCommands.cs.html#1261"><b>1261</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/InternalCommands.cs.html#2302"><b>2302</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/InternalCommands.cs.html#2311"><b>2311</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">engine\lang\parserutils.cs (1)</div>
<a href="../engine/lang/parserutils.cs.html#1963"><b>1963</b><i>var</i> errorRecord = icer.ErrorRecord;</a>
</div>
<div class="rF"><div class="rN">engine\Modules\ExportModuleMemberCommand.cs (2)</div>
<a href="../engine/Modules/ExportModuleMemberCommand.cs.html#160"><b>160</b><i>ErrorRecord</i> er = new ErrorRecord(invalidOp, &quot;Modules_CanOnlyExecuteExportModuleMemberInsideAModule&quot;,</a>
<a href="../engine/Modules/ExportModuleMemberCommand.cs.html#171"><b>171</b><i>var</i> er = new ErrorRecord(se, &quot;Modules_CannotExportMembersAccrossLanguageBoundaries&quot;, ErrorCategory.SecurityError, this);</a>
</div>
<div class="rF"><div class="rN">engine\Modules\GetModuleCommand.cs (5)</div>
<a href="../engine/Modules/GetModuleCommand.cs.html#247"><b>247</b><i>ErrorRecord</i> errorRecord = RemoteDiscoveryHelper.GetErrorRecordForProcessingOfCimModule(e, cimModule.ModuleName);</a>
<a href="../engine/Modules/GetModuleCommand.cs.html#336"><b>336</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/Modules/GetModuleCommand.cs.html#355"><b>355</b><i>ErrorRecord</i> error = new ErrorRecord(new InvalidOperationException(errMsg), &quot;NameAndFullyQualifiedNameCannotBeSpecifiedTogether&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../engine/Modules/GetModuleCommand.cs.html#362"><b>362</b><i>ErrorRecord</i> error = new ErrorRecord(</a>
<a href="../engine/Modules/GetModuleCommand.cs.html#447"><b>447</b><i>var</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">engine\Modules\ImportModuleCommand.cs (11)</div>
<a href="../engine/Modules/ImportModuleCommand.cs.html#472"><b>472</b><i>ErrorRecord</i> er = new ErrorRecord(invalidOp, &quot;Modules_CantUseAsCustomObjectWithBinaryModule&quot;,</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#543"><b>543</b><i>ErrorRecord</i> er = new ErrorRecord(e.ErrorRecord, e);</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#698"><b>698</b><i>ErrorRecord</i> er = new ErrorRecord(invalidOp, &quot;Modules_CantUseAsCustomObjectWithBinaryModule&quot;,</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#819"><b>819</b><i>ErrorRecord</i> er = null;</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#859"><b>859</b><i>ErrorRecord</i> er = new ErrorRecord(e.ErrorRecord, e);</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#1318"><b>1318</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#1338"><b>1338</b><i>ErrorRecord</i> er = new ErrorRecord(fnf, &quot;Modules_ModuleNotFound&quot;,</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#1625"><b>1625</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#1720"><b>1720</b><i>ErrorRecord</i> errorRecord = RemoteDiscoveryHelper.GetErrorRecordForProcessingOfCimModule(e, remoteCimModule.ModuleName);</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#1797"><b>1797</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_GlobalAndScopeParameterCannotBeSpecifiedTogether&quot;,</a>
<a href="../engine/Modules/ImportModuleCommand.cs.html#1939"><b>1939</b><i>ErrorRecord</i> er = new ErrorRecord(exception, &quot;Modules_ModuleInWinCompatDenyList&quot;, ErrorCategory.ResourceUnavailable, exactModuleName);</a>
</div>
<div class="rF"><div class="rN">engine\Modules\ModuleCmdletBase.cs (36)</div>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#459"><b>459</b><i>ErrorRecord</i> er = new ErrorRecord(mm, &quot;Modules_InvalidManifest&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#500"><b>500</b><i>ErrorRecord</i> er = new ErrorRecord(mm, &quot;Modules_InvalidManifest&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#544"><b>544</b><i>ErrorRecord</i> er = new ErrorRecord(ae, &quot;Modules_InvalidManifest&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#619"><b>619</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InsufficientPowerShellVersion&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#636"><b>636</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidManifestMember&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1046"><b>1046</b>private static <i>ErrorRecord</i> CreateModuleNotFoundError(string modulePath)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1050"><b>1050</b><i>ErrorRecord</i> er = new ErrorRecord(fnf, &quot;Modules_ModuleNotFoundForGetModule&quot;, ErrorCategory.ResourceUnavailable, modulePath);</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1416"><b>1416</b>private static <i>ErrorRecord</i> GetErrorRecordIfUnsupportedRootCdxmlAndNestedModuleScenario(</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1443"><b>1443</b><i>ErrorRecord</i> er = new ErrorRecord(</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1544"><b>1544</b><i>ErrorRecord</i> er = new ErrorRecord(ioe,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1674"><b>1674</b><i>ErrorRecord</i> er = new ErrorRecord(mm, &quot;Modules_InvalidManifest&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1725"><b>1725</b><i>var</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidModuleManifestVersion&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1756"><b>1756</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InsufficientPowerShellVersion&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1793"><b>1793</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidPowerShellHostName&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1832"><b>1832</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InsufficientPowerShellHostVersion&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1864"><b>1864</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidProcessorArchitecture&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#1937"><b>1937</b><i>ErrorRecord</i> error = null;</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#2401"><b>2401</b><i>ErrorRecord</i> er = new ErrorRecord(</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#2444"><b>2444</b><i>ErrorRecord</i> er = new ErrorRecord(e, &quot;FormatXmlUpdateException&quot;, ErrorCategory.InvalidOperation,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#2875"><b>2875</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3062"><b>3062</b><i>ErrorRecord</i> errorRecord = GetErrorRecordIfUnsupportedRootCdxmlAndNestedModuleScenario(data,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3661"><b>3661</b><i>ErrorRecord</i> er = GenerateInvalidModuleMemberErrorRecord(manifestElement, moduleManifestPath, e);</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3666"><b>3666</b>private static <i>ErrorRecord</i> GenerateInvalidModuleMemberErrorRecord(string manifestElement, string moduleManifestPath, Exception e)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3670"><b>3670</b><i>ErrorRecord</i> er = new ErrorRecord(newAe, &quot;Modules_InvalidManifest&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3732"><b>3732</b>internal PSModuleInfo LoadRequiredModule(PSModuleInfo currentModule, ModuleSpecification requiredModule, string moduleManifestPath, ManifestProcessingFlags manifestProcessingFlags, bool containedErrors, out <i>ErrorRecord</i> error)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3761"><b>3761</b>out <i>ErrorRecord</i> error)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3933"><b>3933</b>private static PSModuleInfo ImportRequiredModule(ExecutionContext context, ModuleSpecification requiredModule, out <i>ErrorRecord</i> error)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#4146"><b>4146</b>private static bool HasRequiredModulesCyclicReference(ModuleSpecification currentModuleSpecification, List&lt;ModuleSpecification&gt; requiredModules, IEnumerable&lt;PSModuleInfo&gt; moduleInfoList, Dictionary&lt;ModuleSpecification, List&lt;ModuleSpecification&gt;&gt; nonCyclicRequiredModules, out <i>ErrorRecord</i> error)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#4538"><b>4538</b><i>ErrorRecord</i> er = new ErrorRecord(newAe, &quot;Modules_InvalidManifest&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#5108"><b>5108</b><i>ErrorRecord</i> er = new ErrorRecord(memberNotRemoved, &quot;Modules_MemberNotRemoved&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#5267"><b>5267</b><i>ErrorRecord</i> er = new ErrorRecord(invalidOp, &quot;Modules_CantUseAsCustomObjectWithBinaryModule&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#5396"><b>5396</b><i>ErrorRecord</i> er = new ErrorRecord(invalidOp, &quot;Modules_CantUseAsCustomObjectWithBinaryModule&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#5574"><b>5574</b><i>ErrorRecord</i> er = new ErrorRecord(invalidOp, &quot;Modules_ImportPSFileNotAllowedInConstrainedLanguage&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#5898"><b>5898</b><i>ErrorRecord</i> er = new ErrorRecord(invalidOp, &quot;Modules_CantUseAsCustomObjectWithBinaryModule&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#5991"><b>5991</b><i>ErrorRecord</i> er = new ErrorRecord(invalidOp, &quot;Modules_InvalidModuleExtension&quot;,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#6043"><b>6043</b><i>var</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">engine\Modules\ModuleIntrinsics.cs (1)</div>
<a href="../engine/Modules/ModuleIntrinsics.cs.html#63"><b>63</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_ModuleTooDeeplyNested&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\Modules\NewModuleManifestCommand.cs (5)</div>
<a href="../engine/Modules/NewModuleManifestCommand.cs.html#923"><b>923</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidProcessorArchitectureInManifest&quot;,</a>
<a href="../engine/Modules/NewModuleManifestCommand.cs.html#936"><b>936</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidModuleManifestPath&quot;,</a>
<a href="../engine/Modules/NewModuleManifestCommand.cs.html#971"><b>971</b><i>var</i> er = new ErrorRecord(ioe, &quot;Modules_DuplicateEntriesInCompatiblePSEditions&quot;, ErrorCategory.InvalidArgument, CompatiblePSEditions);</a>
<a href="../engine/Modules/NewModuleManifestCommand.cs.html#1159"><b>1159</b><i>var</i> er = new ErrorRecord(ioe, &quot;PrivateDataValueTypeShouldBeHashTable&quot;, ErrorCategory.InvalidArgument, _privateData);</a>
<a href="../engine/Modules/NewModuleManifestCommand.cs.html#1225"><b>1225</b><i>var</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidUri&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\Modules\RemoteDiscoveryHelper.cs (17)</div>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#152"><b>152</b>EventHandler&lt;DataAddedEventArgs&gt; errorHandler = GetStreamForwarder&lt;<i>ErrorRecord</i>&gt;(</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#264"><b>264</b>EventHandler&lt;DataAddedEventArgs&gt; errorHandler = GetStreamForwarder&lt;<i>ErrorRecord</i>&gt;(</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#265"><b>265</b>(<i>ErrorRecord</i> errorRecord) =&gt;</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#310"><b>310</b>internal static <i>ErrorRecord</i> GetErrorRecordForProcessingOfCimModule(Exception innerException, string moduleName)</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#319"><b>319</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(outerException, innerException.GetType().Name, ErrorCategory.NotSpecified, moduleName);</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#325"><b>325</b>private static <i>ErrorRecord</i> GetErrorRecordForRemoteDiscoveryProvider(Exception innerException)</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#339"><b>339</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(outerException, DiscoveryProviderNotFoundErrorId, ErrorCategory.NotImplemented, null);</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#349"><b>349</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(outerException, &quot;DiscoveryProviderFailure&quot;, ErrorCategory.NotSpecified, null);</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#354"><b>354</b>private static <i>ErrorRecord</i> GetErrorRecordForRemotePipelineInvocation(Exception innerException, string errorMessageTemplate)</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#363"><b>363</b><i>ErrorRecord</i> remoteErrorRecord = remoteException?.ErrorRecord;</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#366"><b>366</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(outerException, errorId, errorCategory, null);</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#371"><b>371</b>private static <i>ErrorRecord</i> GetErrorRecordForRemotePipelineInvocation(<i>ErrorRecord</i> innerErrorRecord, string errorMessageTemplate)</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#392"><b>392</b><i>ErrorRecord</i> outerErrorRecord = new ErrorRecord(innerErrorRecord, null /* null means: do not replace the exception */);</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#454"><b>454</b>private static void HandleErrorFromPipeline(Cmdlet cmdlet, <i>ErrorRecord</i> errorRecord, PowerShell powerShell)</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#491"><b>491</b><i>ErrorRecord</i> errorRecord = GetErrorRecordForRemotePipelineInvocation(exception, errorMessageTemplate);</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#815"><b>815</b><i>ErrorRecord</i> errorRecord = GetErrorRecordForRemoteDiscoveryProvider(exception);</a>
</div>
<div class="rF"><div class="rN">engine\Modules\RemoveModuleCommand.cs (4)</div>
<a href="../engine/Modules/RemoveModuleCommand.cs.html#162"><b>162</b><i>ErrorRecord</i> er = new ErrorRecord(moduleNotRemoved, &quot;Modules_ModuleIsConstant&quot;,</a>
<a href="../engine/Modules/RemoveModuleCommand.cs.html#180"><b>180</b><i>ErrorRecord</i> er = new ErrorRecord(moduleNotRemoved, &quot;Modules_ModuleIsReadOnly&quot;,</a>
<a href="../engine/Modules/RemoveModuleCommand.cs.html#251"><b>251</b><i>ErrorRecord</i> er = new ErrorRecord(moduleNotRemoved, &quot;Modules_ModuleIsRequired&quot;,</a>
<a href="../engine/Modules/RemoveModuleCommand.cs.html#374"><b>374</b><i>ErrorRecord</i> er = new ErrorRecord(invalidOp, &quot;Modules_NoModulesRemoved&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\Modules\TestModuleManifestCommand.cs (12)</div>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#78"><b>78</b><i>ErrorRecord</i> er = new ErrorRecord(fnf, &quot;Modules_ModuleNotFound&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#97"><b>97</b><i>ErrorRecord</i> er = new ErrorRecord(fnf, &quot;Modules_ModuleNotFound&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#142"><b>142</b><i>var</i> errorRecord = new ErrorRecord(new DirectoryNotFoundException(errorMsg), &quot;Modules_InvalidRequiredAssembliesInModuleManifest&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#152"><b>152</b><i>var</i> errorRecord = new ErrorRecord(new ArgumentException(errorMsg), &quot;Modules_InvalidRootModuleInModuleManifest&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#178"><b>178</b><i>var</i> errorRecord = new ErrorRecord(new DirectoryNotFoundException(errorMsg), &quot;Modules_InvalidNestedModuleinModuleManifest&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#196"><b>196</b><i>var</i> errorRecord = new ErrorRecord(new DirectoryNotFoundException(errorMsg), &quot;Modules_InvalidRequiredModulesinModuleManifest&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#212"><b>212</b><i>var</i> errorRecord = new ErrorRecord(new DirectoryNotFoundException(errorMsg), &quot;Modules_InvalidFilePathinModuleManifest&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#229"><b>229</b><i>var</i> errorRecord = new ErrorRecord(new DirectoryNotFoundException(errorMsg), &quot;Modules_InvalidModuleListinModuleManifest&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#245"><b>245</b><i>var</i> errorRecord = new ErrorRecord(new ArgumentException(errorMsg), &quot;Modules_InvalidPowerShellVersionInModuleManifest&quot;, ErrorCategory.InvalidArgument, _path);</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#272"><b>272</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidModuleManifestVersion&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#289"><b>289</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidModuleManifestPath&quot;,</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#373"><b>373</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;Modules_InvalidModuleManifestPath&quot;, ErrorCategory.InvalidArgument, path);</a>
</div>
<div class="rF"><div class="rN">engine\MshCommandRuntime.cs (11)</div>
<a href="../engine/MshCommandRuntime.cs.html#163"><b>163</b>/// &lt;seealso cref=&quot;System.Management.Automation.ICommandRuntime.WriteError(<i>ErrorRecord</i>)&quot;/&gt;</a>
<a href="../engine/MshCommandRuntime.cs.html#235"><b>235</b>/// &lt;seealso cref=&quot;System.Management.Automation.ICommandRuntime.WriteError(<i>ErrorRecord</i>)&quot;/&gt;</a>
<a href="../engine/MshCommandRuntime.cs.html#2046"><b>2046</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>&quot;/&gt;</a>
<a href="../engine/MshCommandRuntime.cs.html#2059"><b>2059</b>public void ThrowTerminatingError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/MshCommandRuntime.cs.html#2708"><b>2708</b>public void WriteError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/MshCommandRuntime.cs.html#2713"><b>2713</b>internal void WriteError(<i>ErrorRecord</i> errorRecord, bool overrideInquire)</a>
<a href="../engine/MshCommandRuntime.cs.html#2735"><b>2735</b>DoWriteError(new KeyValuePair&lt;<i>ErrorRecord</i>, ActionPreference&gt;(errorRecord, preference));</a>
<a href="../engine/MshCommandRuntime.cs.html#2774"><b>2774</b>KeyValuePair&lt;<i>ErrorRecord</i>, ActionPreference&gt; pair = (KeyValuePair&lt;<i>ErrorRecord</i>, ActionPreference&gt;)obj;</a>
<a href="../engine/MshCommandRuntime.cs.html#2775"><b>2775</b><i>ErrorRecord</i> errorRecord = pair.Key;</a>
<a href="../engine/MshCommandRuntime.cs.html#2830"><b>2830</b>internal void _WriteErrorSkipAllowCheck(<i>ErrorRecord</i> errorRecord, ActionPreference? actionPreference = null, bool isFromNativeStdError = false)</a>
</div>
<div class="rF"><div class="rN">engine\MshSecurityException.cs (2)</div>
<a href="../engine/MshSecurityException.cs.html#92"><b>92</b>public override <i>ErrorRecord</i> ErrorRecord</a>
<a href="../engine/MshSecurityException.cs.html#109"><b>109</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
<div class="rF"><div class="rN">engine\NativeCommandProcessor.cs (9)</div>
<a href="../engine/NativeCommandProcessor.cs.html#887"><b>887</b><i>var</i> errorRecord = new ErrorRecord(exception, errorId, ErrorCategory.NotSpecified, targetObject: NativeCommandName);</a>
<a href="../engine/NativeCommandProcessor.cs.html#1209"><b>1209</b><i>ErrorRecord</i> record = outputValue.Data as <i>ErrorRecord</i>;</a>
<a href="../engine/NativeCommandProcessor.cs.html#1812"><b>1812</b><i>ErrorRecord</i> errorRecord;</a>
<a href="../engine/NativeCommandProcessor.cs.html#1870"><b>1870</b>obj = <i>ErrorRecord</i>.FromPSObjectForRemoting(PSObject.AsPSObject(obj));</a>
<a href="../engine/NativeCommandProcessor.cs.html#1941"><b>1941</b><i>ErrorRecord</i> error = new ErrorRecord(</a>
<a href="../engine/NativeCommandProcessor.cs.html#2425"><b>2425</b>private <i>ErrorRecord</i> _remoteErrorRecord;</a>
<a href="../engine/NativeCommandProcessor.cs.html#2430"><b>2430</b>internal void SetRemoteErrorRecord(<i>ErrorRecord</i> remoteError)</a>
<a href="../engine/NativeCommandProcessor.cs.html#2438"><b>2438</b>public override <i>ErrorRecord</i> ErrorRecord</a>
</div>
<div class="rF"><div class="rN">engine\ObjectEventRegistrationBase.cs (3)</div>
<a href="../engine/ObjectEventRegistrationBase.cs.html#187"><b>187</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/ObjectEventRegistrationBase.cs.html#212"><b>212</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/ObjectEventRegistrationBase.cs.html#222"><b>222</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">engine\parser\Compiler.cs (3)</div>
<a href="../engine/parser/Compiler.cs.html#151"><b>151</b>typeof(<i>ErrorRecord</i>).GetConstructor(</a>
<a href="../engine/parser/Compiler.cs.html#155"><b>155</b>new Type[] { typeof(<i>ErrorRecord</i>), typeof(RuntimeException) },</a>
<a href="../engine/parser/Compiler.cs.html#2856"><b>2856</b><i>var</i> errorRecord = ps.Streams.Error[0];</a>
</div>
<div class="rF"><div class="rN">engine\parser\TypeInferenceVisitor.cs (1)</div>
<a href="../engine/parser/TypeInferenceVisitor.cs.html#1832"><b>1832</b>inferredTypes.Add(new PSTypeName(typeof(<i>ErrorRecord</i>)));</a>
</div>
<div class="rF"><div class="rN">engine\pipeline.cs (1)</div>
<a href="../engine/pipeline.cs.html#141"><b>141</b>internal void LogExecutionError(InvocationInfo invocationInfo, <i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">engine\regex.cs (3)</div>
<a href="../engine/regex.cs.html#447"><b>447</b>internal WildcardPatternException(<i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/regex.cs.html#459"><b>459</b>private readonly <i>ErrorRecord</i> _errorRecord;</a>
<a href="../engine/regex.cs.html#721"><b>721</b><i>ErrorRecord</i> er =</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\ClientMethodExecutor.cs (8)</div>
<a href="../engine/remoting/client/ClientMethodExecutor.cs.html#74"><b>74</b>PSDataCollectionStream&lt;<i>ErrorRecord</i>&gt; errorStream,</a>
<a href="../engine/remoting/client/ClientMethodExecutor.cs.html#151"><b>151</b>internal void Execute(PSDataCollectionStream&lt;<i>ErrorRecord</i>&gt; errorStream)</a>
<a href="../engine/remoting/client/ClientMethodExecutor.cs.html#153"><b>153</b>Action&lt;<i>ErrorRecord</i>&gt; writeErrorAction = null;</a>
<a href="../engine/remoting/client/ClientMethodExecutor.cs.html#158"><b>158</b>writeErrorAction = (<i>ErrorRecord</i> errorRecord) =&gt;</a>
<a href="../engine/remoting/client/ClientMethodExecutor.cs.html#177"><b>177</b>writeErrorAction = (<i>ErrorRecord</i> errorRecord) =&gt; errorStream.Write(errorRecord);</a>
<a href="../engine/remoting/client/ClientMethodExecutor.cs.html#194"><b>194</b>internal void Execute(Action&lt;<i>ErrorRecord</i>&gt; writeErrorAction)</a>
<a href="../engine/remoting/client/ClientMethodExecutor.cs.html#217"><b>217</b>internal void ExecuteVoid(Action&lt;<i>ErrorRecord</i>&gt; writeErrorAction)</a>
<a href="../engine/remoting/client/ClientMethodExecutor.cs.html#234"><b>234</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\ClientRemotePowerShell.cs (3)</div>
<a href="../engine/remoting/client/ClientRemotePowerShell.cs.html#360"><b>360</b>private void HandleErrorReceived(object sender, RemoteDataEventArgs&lt;<i>ErrorRecord</i>&gt; eventArgs)</a>
<a href="../engine/remoting/client/ClientRemotePowerShell.cs.html#779"><b>779</b><i>ErrorRecord</i> errorRecord = null;</a>
<a href="../engine/remoting/client/ClientRemotePowerShell.cs.html#896"><b>896</b>new RemoteDataEventArgs&lt;<i>ErrorRecord</i>&gt;(remotingErrorRecord));</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\Job.cs (18)</div>
<a href="../engine/remoting/client/Job.cs.html#533"><b>533</b>private PSDataCollection&lt;<i>ErrorRecord</i>&gt; _error = new PSDataCollection&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../engine/remoting/client/Job.cs.html#743"><b>743</b>private void WriteError(Cmdlet cmdlet, <i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/remoting/client/Job.cs.html#756"><b>756</b>private static Exception GetExceptionFromErrorRecord(<i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/remoting/client/Job.cs.html#773"><b>773</b>internal virtual void WriteError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/remoting/client/Job.cs.html#789"><b>789</b>internal void WriteError(<i>ErrorRecord</i> errorRecord, out Exception exceptionThrownOnCmdletThread)</a>
<a href="../engine/remoting/client/Job.cs.html#1120"><b>1120</b>public PSDataCollection&lt;<i>ErrorRecord</i>&gt; Error</a>
<a href="../engine/remoting/client/Job.cs.html#2075"><b>2075</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;PSJobConnectFailed&quot;, ErrorCategory.InvalidOperation, _psRemoteChildJob);</a>
<a href="../engine/remoting/client/Job.cs.html#2261"><b>2261</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;StopJobCannotConnectToServer&quot;,</a>
<a href="../engine/remoting/client/Job.cs.html#3086"><b>3086</b>PSDataCollectionPipelineReader&lt;<i>ErrorRecord</i>, object&gt; reader =</a>
<a href="../engine/remoting/client/Job.cs.html#3087"><b>3087</b>sender as PSDataCollectionPipelineReader&lt;<i>ErrorRecord</i>, object&gt;;</a>
<a href="../engine/remoting/client/Job.cs.html#3093"><b>3093</b><i>ErrorRecord</i> er = errorData as <i>ErrorRecord</i>;</a>
<a href="../engine/remoting/client/Job.cs.html#3263"><b>3263</b>private <i>ErrorRecord</i> _failureErrorRecord;</a>
<a href="../engine/remoting/client/Job.cs.html#3269"><b>3269</b>internal <i>ErrorRecord</i> FailureErrorRecord</a>
<a href="../engine/remoting/client/Job.cs.html#3284"><b>3284</b>out <i>ErrorRecord</i> failureErrorRecord)</a>
<a href="../engine/remoting/client/Job.cs.html#3398"><b>3398</b><i>ErrorRecord</i> errorRecord = null;</a>
<a href="../engine/remoting/client/Job.cs.html#4292"><b>4292</b><i>ErrorRecord</i> failureErrorRecord;</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\Job2.cs (3)</div>
<a href="../engine/remoting/client/Job2.cs.html#530"><b>530</b>private readonly PSDataCollection&lt;<i>ErrorRecord</i>&gt; _executionError = new PSDataCollection&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../engine/remoting/client/Job2.cs.html#697"><b>697</b>internal PSDataCollection&lt;<i>ErrorRecord</i>&gt; ExecutionError { get { return _executionError; } }</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\PowerShellStreams.cs (3)</div>
<a href="../engine/remoting/client/PowerShellStreams.cs.html#24"><b>24</b>private PSDataCollection&lt;<i>ErrorRecord</i>&gt; _errorStream;</a>
<a href="../engine/remoting/client/PowerShellStreams.cs.html#89"><b>89</b>_errorStream = new PSDataCollection&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../engine/remoting/client/PowerShellStreams.cs.html#173"><b>173</b>public PSDataCollection&lt;<i>ErrorRecord</i>&gt; ErrorStream</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\remotepipeline.cs (4)</div>
<a href="../engine/remoting/client/remotepipeline.cs.html#88"><b>88</b>_errorCollection = new PSDataCollection&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../engine/remoting/client/remotepipeline.cs.html#89"><b>89</b>_errorStream = new PSDataCollectionStream&lt;<i>ErrorRecord</i>&gt;(Guid.Empty, _errorCollection);</a>
<a href="../engine/remoting/client/remotepipeline.cs.html#369"><b>369</b>private readonly PSDataCollection&lt;<i>ErrorRecord</i>&gt; _errorCollection;</a>
<a href="../engine/remoting/client/remotepipeline.cs.html#370"><b>370</b>private readonly PSDataCollectionStream&lt;<i>ErrorRecord</i>&gt; _errorStream;</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\remoterunspace.cs (4)</div>
<a href="../engine/remoting/client/remoterunspace.cs.html#3003"><b>3003</b><i>ErrorRecord</i> error = (<i>ErrorRecord</i>)remotePipeline.Error.Read();</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#3063"><b>3063</b><i>ErrorRecord</i> error = (<i>ErrorRecord</i>)remotePipeline.Error.Read();</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\RemotingErrorRecord.cs (3)</div>
<a href="../engine/remoting/client/RemotingErrorRecord.cs.html#35"><b>35</b>public RemotingErrorRecord(<i>ErrorRecord</i> errorRecord, OriginInfo originInfo)</a>
<a href="../engine/remoting/client/RemotingErrorRecord.cs.html#45"><b>45</b><i>ErrorRecord</i> errorRecord,</a>
<a href="../engine/remoting/client/RemotingErrorRecord.cs.html#100"><b>100</b>internal override <i>ErrorRecord</i> WrapException(Exception replaceParentContainsErrorRecordException)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\RemotingProtocol2.cs (3)</div>
<a href="../engine/remoting/client/RemotingProtocol2.cs.html#1009"><b>1009</b>internal event EventHandler&lt;RemoteDataEventArgs&lt;<i>ErrorRecord</i>&gt;&gt; ErrorReceived;</a>
<a href="../engine/remoting/client/RemotingProtocol2.cs.html#1250"><b>1250</b><i>ErrorRecord</i> errorRecord =</a>
<a href="../engine/remoting/client/RemotingProtocol2.cs.html#1264"><b>1264</b>new RemoteDataEventArgs&lt;<i>ErrorRecord</i>&gt;(errorRecord));</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\RunspaceRef.cs (3)</div>
<a href="../engine/remoting/client/RunspaceRef.cs.html#232"><b>232</b>PSDataCollection&lt;<i>ErrorRecord</i>&gt; erBuffer = sender as PSDataCollection&lt;<i>ErrorRecord</i>&gt;;</a>
<a href="../engine/remoting/client/RunspaceRef.cs.html#236"><b>236</b>Collection&lt;<i>ErrorRecord</i>&gt; erRecords = erBuffer.ReadAll();</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ConnectPSSession.cs (3)</div>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#742"><b>742</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, FQEID, ErrorCategory.InvalidOperation, null);</a>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#866"><b>866</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;CannotConnectVMContainerSession&quot;, ErrorCategory.InvalidOperation, psSession);</a>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#889"><b>889</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;PSSessionConnectFailed&quot;, ErrorCategory.InvalidOperation, psSession);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\CustomShellCommands.cs (10)</div>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#814"><b>814</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;InvalidPSSessionConfigurationFilePath&quot;,</a>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#835"><b>835</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;InvalidPSSessionConfigurationFilePath&quot;,</a>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#844"><b>844</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;InvalidPSSessionConfigurationFile&quot;,</a>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#911"><b>911</b><i>ErrorRecord</i> error;</a>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#967"><b>967</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;InvalidProcessorArchitecture&quot;, ErrorCategory.InvalidArgument, Path);</a>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#1505"><b>1505</b>out <i>ErrorRecord</i> error)</a>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#3311"><b>3311</b><i>ErrorRecord</i> error;</a>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#3948"><b>3948</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;InvalidPSSessionConfigurationFilePath&quot;,</a>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#5299"><b>5299</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;CouldNotResolveUsername&quot;, ErrorCategory.InvalidArgument, this.Username);</a>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#5321"><b>5321</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;SessionConfigurationMustBeFileBased&quot;, ErrorCategory.InvalidArgument, foundConfiguration);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\DisconnectPSSession.cs (4)</div>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#217"><b>217</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;CannotDisconnectVMContainerSession&quot;, ErrorCategory.InvalidOperation, psSession);</a>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#246"><b>246</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;CannotDisconnectSessionWhenNotOpened&quot;, ErrorCategory.InvalidOperation, psSession);</a>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#356"><b>356</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new RuntimeException(msg),</a>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#505"><b>505</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;PSSessionDisconnectFailed&quot;, ErrorCategory.InvalidOperation, _remoteSession);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\InvokeCommandCommand.cs (2)</div>
<a href="../engine/remoting/commands/InvokeCommandCommand.cs.html#1983"><b>1983</b><i>ErrorRecord</i> errorRecord = streamObject.Value as <i>ErrorRecord</i>;</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\NewPSSessionConfigurationFile.cs (2)</div>
<a href="../engine/remoting/commands/NewPSSessionConfigurationFile.cs.html#653"><b>653</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;InvalidPSSessionConfigurationFilePath&quot;,</a>
<a href="../engine/remoting/commands/NewPSSessionConfigurationFile.cs.html#1555"><b>1555</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;InvalidRoleCapabilityFilePath&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\newrunspacecommand.cs (10)</div>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#541"><b>541</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#600"><b>600</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#707"><b>707</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e, &quot;CreateRemoteRunspaceFailed&quot;,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#841"><b>841</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e, &quot;CreateRemoteRunspaceFailed&quot;,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#973"><b>973</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#982"><b>982</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#1038"><b>1038</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#1048"><b>1048</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#1058"><b>1058</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#1249"><b>1249</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e, &quot;CreateRemoteRunspaceFailed&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\PSRemotingCmdlet.cs (10)</div>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#1438"><b>1438</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(uriException, &quot;CreateRemoteRunspaceFailed&quot;,</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#1765"><b>1765</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#1774"><b>1774</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#1830"><b>1830</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#1840"><b>1840</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#1850"><b>1850</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#1990"><b>1990</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e, &quot;CreateRemoteRunspaceFailed&quot;,</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#3921"><b>3921</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, FQEID, ErrorCategory.InvalidOperation, connectionInfo);</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#4014"><b>4014</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;PSSessionIdMatchFail&quot;, ErrorCategory.InvalidOperation, id);</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#4047"><b>4047</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;PSSessionNameMatchFail&quot;, ErrorCategory.InvalidOperation, name);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\PushRunspaceCommand.cs (9)</div>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#627"><b>627</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(exception, argument,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1029"><b>1029</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1038"><b>1038</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1047"><b>1047</b><i>ErrorRecord</i> errorRecord;</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1072"><b>1072</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1205"><b>1205</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1214"><b>1214</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1223"><b>1223</b><i>ErrorRecord</i> errorRecord;</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#1248"><b>1248</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ReceiveJob.cs (8)</div>
<a href="../engine/remoting/commands/ReceiveJob.cs.html#807"><b>807</b>Collection&lt;<i>ErrorRecord</i>&gt; errorRecords = ReadAll&lt;<i>ErrorRecord</i>&gt;(job.Error);</a>
<a href="../engine/remoting/commands/ReceiveJob.cs.html#809"><b>809</b>foreach (<i>ErrorRecord</i> e in errorRecords)</a>
<a href="../engine/remoting/commands/ReceiveJob.cs.html#907"><b>907</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(resultReason, &quot;JobStateFailed&quot;, ErrorCategory.InvalidResult, null);</a>
<a href="../engine/remoting/commands/ReceiveJob.cs.html#1192"><b>1192</b>PSDataCollection&lt;<i>ErrorRecord</i>&gt; errorRecords = sender as PSDataCollection&lt;<i>ErrorRecord</i>&gt;;</a>
<a href="../engine/remoting/commands/ReceiveJob.cs.html#1194"><b>1194</b><i>ErrorRecord</i> errorRecord = GetData(errorRecords, e.Index);</a>
<a href="../engine/remoting/commands/ReceiveJob.cs.html#1450"><b>1450</b><i>var</i> removeError = new ErrorRecord(ex2, &quot;ReceiveJobAutoRemovalError&quot;, ErrorCategory.InvalidOperation, job);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ReceivePSSession.cs (9)</div>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#447"><b>447</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, FQEID, ErrorCategory.InvalidOperation, connectionInfo);</a>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#702"><b>702</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;CannotReceiveVMContainerSession&quot;, ErrorCategory.InvalidOperation, session);</a>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#978"><b>978</b>if (errorObj is Collection&lt;<i>ErrorRecord</i>&gt;)</a>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#980"><b>980</b>Collection&lt;<i>ErrorRecord</i>&gt; errorCollection = (Collection&lt;<i>ErrorRecord</i>&gt;)errorObj;</a>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#981"><b>981</b>foreach (<i>ErrorRecord</i> errorRecord in errorCollection)</a>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#986"><b>986</b>else if (errorObj is <i>ErrorRecord</i>)</a>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#988"><b>988</b>WriteError((<i>ErrorRecord</i>)errorObj);</a>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#1026"><b>1026</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new RuntimeException(msg, reason),</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\remotingcommandutil.cs (1)</div>
<a href="../engine/remoting/commands/remotingcommandutil.cs.html#164"><b>164</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(RemotingErrorIdStrings.WinPERemotingNotSupported)), null, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\removerunspacecommand.cs (1)</div>
<a href="../engine/remoting/commands/removerunspacecommand.cs.html#161"><b>161</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(reason, &quot;RemoveSessionCannotConnectToServer&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\StartJob.cs (3)</div>
<a href="../engine/remoting/commands/StartJob.cs.html#580"><b>580</b><i>var</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/remoting/commands/StartJob.cs.html#593"><b>593</b><i>var</i> errorRecord = new ErrorRecord(</a>
<a href="../engine/remoting/commands/StartJob.cs.html#605"><b>605</b><i>var</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\StopJob.cs (2)</div>
<a href="../engine/remoting/commands/StopJob.cs.html#261"><b>261</b>private readonly List&lt;<i>ErrorRecord</i>&gt; _errorsToWrite = new List&lt;<i>ErrorRecord</i>&gt;();</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\TestPSSessionConfigurationFile.cs (3)</div>
<a href="../engine/remoting/commands/TestPSSessionConfigurationFile.cs.html#65"><b>65</b><i>ErrorRecord</i> er = new ErrorRecord(fnf, &quot;PSSessionConfigurationFileNotFound&quot;,</a>
<a href="../engine/remoting/commands/TestPSSessionConfigurationFile.cs.html#84"><b>84</b><i>ErrorRecord</i> er = new ErrorRecord(fnf, &quot;PSSessionConfigurationFileNotFound&quot;,</a>
<a href="../engine/remoting/commands/TestPSSessionConfigurationFile.cs.html#132"><b>132</b><i>ErrorRecord</i> er = new ErrorRecord(ioe, &quot;InvalidPSSessionConfigurationFilePath&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\WaitJob.cs (1)</div>
<a href="../engine/remoting/commands/WaitJob.cs.html#378"><b>378</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\common\psstreamobject.cs (6)</div>
<a href="../engine/remoting/common/psstreamobject.cs.html#120"><b>120</b><i>ErrorRecord</i> errorRecord = (<i>ErrorRecord</i>)this.Value;</a>
<a href="../engine/remoting/common/psstreamobject.cs.html#288"><b>288</b><i>ErrorRecord</i> errorRecord = (<i>ErrorRecord</i>)this.Value;</a>
<a href="../engine/remoting/common/psstreamobject.cs.html#499"><b>499</b>internal static <i>ErrorRecord</i> AddSourceTagToError(<i>ErrorRecord</i> errorRecord, Guid sourceId)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\common\WireDataFormat\EncodeAndDecode.cs (8)</div>
<a href="../engine/remoting/common/WireDataFormat/EncodeAndDecode.cs.html#1545"><b>1545</b>internal static <i>ErrorRecord</i> GetErrorRecordFromException(Exception exception)</a>
<a href="../engine/remoting/common/WireDataFormat/EncodeAndDecode.cs.html#1549"><b>1549</b><i>ErrorRecord</i> er = null;</a>
<a href="../engine/remoting/common/WireDataFormat/EncodeAndDecode.cs.html#1576"><b>1576</b><i>ErrorRecord</i> er = GetErrorRecordFromException(exception) ??</a>
<a href="../engine/remoting/common/WireDataFormat/EncodeAndDecode.cs.html#2061"><b>2061</b><i>ErrorRecord</i> er = <i>ErrorRecord</i>.FromPSObjectForRemoting(PSObject.AsPSObject(serializedErrorRecord));</a>
<a href="../engine/remoting/common/WireDataFormat/EncodeAndDecode.cs.html#2109"><b>2109</b>internal static <i>ErrorRecord</i> GetPowerShellError(object data)</a>
<a href="../engine/remoting/common/WireDataFormat/EncodeAndDecode.cs.html#2118"><b>2118</b><i>ErrorRecord</i> errorRecord = <i>ErrorRecord</i>.FromPSObjectForRemoting(dataAsPSObject);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\common\WireDataFormat\RemoteHostEncoder.cs (3)</div>
<a href="../engine/remoting/common/WireDataFormat/RemoteHostEncoder.cs.html#258"><b>258</b><i>ErrorRecord</i> errorRecord = null;</a>
<a href="../engine/remoting/common/WireDataFormat/RemoteHostEncoder.cs.html#283"><b>283</b><i>ErrorRecord</i> errorRecord = <i>ErrorRecord</i>.FromPSObjectForRemoting(psObject);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\server\ServerPowerShellDriver.cs (2)</div>
<a href="../engine/remoting/server/ServerPowerShellDriver.cs.html#529"><b>529</b><i>ErrorRecord</i> errorRecord = LocalPowerShell.Streams.Error[index];</a>
<a href="../engine/remoting/server/ServerPowerShellDriver.cs.html#699"><b>699</b><i>ErrorRecord</i> errorRecord = LocalPowerShell.Streams.Error[i];</a>
</div>
<div class="rF"><div class="rN">engine\remoting\server\ServerRemotingProtocol2.cs (1)</div>
<a href="../engine/remoting/server/ServerRemotingProtocol2.cs.html#537"><b>537</b>internal void SendErrorRecordToClient(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\server\ServerRunspacePoolDriver.cs (2)</div>
<a href="../engine/remoting/server/ServerRunspacePoolDriver.cs.html#554"><b>554</b><i>ErrorRecord</i> lastErrorRecord = errorList[0] as <i>ErrorRecord</i>;</a>
</div>
<div class="rF"><div class="rN">engine\runtime\CompiledScriptBlock.cs (2)</div>
<a href="../engine/runtime/CompiledScriptBlock.cs.html#1494"><b>1494</b><i>ErrorRecord</i> error = null;</a>
<a href="../engine/runtime/CompiledScriptBlock.cs.html#1624"><b>1624</b><i>ErrorRecord</i> error = null;</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Operations\MiscOps.cs (8)</div>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1395"><b>1395</b>internal <i>ErrorRecord</i> ErrorRecordToPass;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1444"><b>1444</b><i>ErrorRecord</i> errorRecord)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1487"><b>1487</b><i>ErrorRecord</i> errorRecordToPass = rte.ErrorRecord;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1538"><b>1538</b><i>var</i> errorRecord = new ErrorRecord(current.ErrorRecordToPass, current.ExceptionToPass);</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1726"><b>1726</b><i>ErrorRecord</i> err = rte.ErrorRecord;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1922"><b>1922</b><i>ErrorRecord</i> errRec = rte.ErrorRecord.WrapException(rte);</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1992"><b>1992</b><i>ErrorRecord</i> er = result as <i>ErrorRecord</i>;</a>
</div>
<div class="rF"><div class="rN">engine\serialization.cs (2)</div>
<a href="../engine/serialization.cs.html#1474"><b>1474</b><i>ErrorRecord</i> errorRecord = mshSource.ImmediateBaseObject as <i>ErrorRecord</i>;</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\BaseCommand.cs (1)</div>
<a href="../FormatAndOutput/common/BaseCommand.cs.html#26"><b>26</b>internal void ThrowTerminatingError(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\BaseFormattingCommand.cs (6)</div>
<a href="../FormatAndOutput/common/BaseFormattingCommand.cs.html#321"><b>321</b>List&lt;<i>ErrorRecord</i>&gt; errors;</a>
<a href="../FormatAndOutput/common/BaseFormattingCommand.cs.html#353"><b>353</b>List&lt;<i>ErrorRecord</i>&gt; errors;</a>
<a href="../FormatAndOutput/common/BaseFormattingCommand.cs.html#391"><b>391</b>private void WriteErrorRecords(List&lt;<i>ErrorRecord</i>&gt; errorRecordList)</a>
<a href="../FormatAndOutput/common/BaseFormattingCommand.cs.html#401"><b>401</b>foreach (<i>ErrorRecord</i> errorRecord in errorRecordList)</a>
<a href="../FormatAndOutput/common/BaseFormattingCommand.cs.html#489"><b>489</b>List&lt;<i>ErrorRecord</i>&gt; errors = _viewManager.ViewGenerator.ErrorManager.DrainFailedResultList();</a>
<a href="../FormatAndOutput/common/BaseFormattingCommand.cs.html#706"><b>706</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\BaseOutputtingCommand.cs (1)</div>
<a href="../FormatAndOutput/common/BaseOutputtingCommand.cs.html#352"><b>352</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\FormattingObjectsDeserializer.cs (6)</div>
<a href="../FormatAndOutput/common/FormattingObjectsDeserializer.cs.html#140"><b>140</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../FormatAndOutput/common/FormattingObjectsDeserializer.cs.html#215"><b>215</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../FormatAndOutput/common/FormattingObjectsDeserializer.cs.html#245"><b>245</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../FormatAndOutput/common/FormattingObjectsDeserializer.cs.html#341"><b>341</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../FormatAndOutput/common/FormattingObjectsDeserializer.cs.html#400"><b>400</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../FormatAndOutput/common/FormattingObjectsDeserializer.cs.html#472"><b>472</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\FormatViewManager.cs (8)</div>
<a href="../FormatAndOutput/common/FormatViewManager.cs.html#350"><b>350</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../FormatAndOutput/common/FormatViewManager.cs.html#482"><b>482</b>TypeInfoDataBase db, PSObject so, int enumerationLimit, bool useToStringFallback, out List&lt;<i>ErrorRecord</i>&gt; errors)</a>
<a href="../FormatAndOutput/common/FormatViewManager.cs.html#623"><b>623</b>internal List&lt;<i>ErrorRecord</i>&gt; DrainFailedResultList()</a>
<a href="../FormatAndOutput/common/FormatViewManager.cs.html#628"><b>628</b>List&lt;<i>ErrorRecord</i>&gt; retVal = new List&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../FormatAndOutput/common/FormatViewManager.cs.html#631"><b>631</b><i>ErrorRecord</i> errorRecord = GenerateErrorRecord(error);</a>
<a href="../FormatAndOutput/common/FormatViewManager.cs.html#645"><b>645</b>private static <i>ErrorRecord</i> GenerateErrorRecord(FormattingError error)</a>
<a href="../FormatAndOutput/common/FormatViewManager.cs.html#647"><b>647</b><i>ErrorRecord</i> errorRecord = null;</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\Utilities\MshParameter.cs (1)</div>
<a href="../FormatAndOutput/common/Utilities/MshParameter.cs.html#238"><b>238</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\out-console\OutConsole.cs (1)</div>
<a href="../FormatAndOutput/out-console/OutConsole.cs.html#113"><b>113</b>inputObjectBase is not <i>ErrorRecord</i> &amp;&amp;</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\out-textInterface\OutTextInterface.cs (2)</div>
<a href="../FormatAndOutput/out-textInterface/OutTextInterface.cs.html#66"><b>66</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../FormatAndOutput/out-textInterface/OutTextInterface.cs.html#82"><b>82</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">help\HelpCategoryInvalidException.cs (2)</div>
<a href="../help/HelpCategoryInvalidException.cs.html#65"><b>65</b>private <i>ErrorRecord</i> _errorRecord;</a>
<a href="../help/HelpCategoryInvalidException.cs.html#71"><b>71</b>public <i>ErrorRecord</i> ErrorRecord</a>
</div>
<div class="rF"><div class="rN">help\HelpCommands.cs (3)</div>
<a href="../help/HelpCommands.cs.html#348"><b>348</b>foreach (<i>ErrorRecord</i> errorRecord in helpSystem.LastErrors)</a>
<a href="../help/HelpCommands.cs.html#385"><b>385</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e, &quot;InvalidHelpCategory&quot;, ErrorCategory.InvalidArgument, null);</a>
<a href="../help/HelpCommands.cs.html#716"><b>716</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e, &quot;ParameterNotValidInRemoteRunspace&quot;, ErrorCategory.InvalidArgument, null);</a>
</div>
<div class="rF"><div class="rN">help\HelpErrorTracer.cs (9)</div>
<a href="../help/HelpErrorTracer.cs.html#58"><b>58</b>private readonly Collection&lt;<i>ErrorRecord</i>&gt; _errors = new Collection&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../help/HelpErrorTracer.cs.html#77"><b>77</b>internal void TraceError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../help/HelpErrorTracer.cs.html#88"><b>88</b>internal void TraceErrors(Collection&lt;<i>ErrorRecord</i>&gt; errorRecords)</a>
<a href="../help/HelpErrorTracer.cs.html#92"><b>92</b>foreach (<i>ErrorRecord</i> errorRecord in errorRecords)</a>
<a href="../help/HelpErrorTracer.cs.html#107"><b>107</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new ParentContainsErrorRecordException(&quot;Help Load Error&quot;), &quot;HelpLoadError&quot;, ErrorCategory.SyntaxError, null);</a>
<a href="../help/HelpErrorTracer.cs.html#111"><b>111</b>foreach (<i>ErrorRecord</i> error in _errors)</a>
<a href="../help/HelpErrorTracer.cs.html#157"><b>157</b>internal void TraceError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../help/HelpErrorTracer.cs.html#172"><b>172</b>internal void TraceErrors(Collection&lt;<i>ErrorRecord</i>&gt; errorRecords)</a>
</div>
<div class="rF"><div class="rN">help\HelpInfo.cs (1)</div>
<a href="../help/HelpInfo.cs.html#265"><b>265</b>internal Collection&lt;<i>ErrorRecord</i>&gt; Errors { get; set; }</a>
</div>
<div class="rF"><div class="rN">help\HelpNotFoundException.cs (2)</div>
<a href="../help/HelpNotFoundException.cs.html#70"><b>70</b>private <i>ErrorRecord</i> _errorRecord;</a>
<a href="../help/HelpNotFoundException.cs.html#76"><b>76</b>public <i>ErrorRecord</i> ErrorRecord</a>
</div>
<div class="rF"><div class="rN">help\HelpProvider.cs (1)</div>
<a href="../help/HelpProvider.cs.html#213"><b>213</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(exception, &quot;LoadHelpFileForTargetFailed&quot;, ErrorCategory.OpenError, null);</a>
</div>
<div class="rF"><div class="rN">help\HelpSystem.cs (6)</div>
<a href="../help/HelpSystem.cs.html#174"><b>174</b>private readonly Collection&lt;<i>ErrorRecord</i>&gt; _lastErrors = new Collection&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../help/HelpSystem.cs.html#181"><b>181</b>internal Collection&lt;<i>ErrorRecord</i>&gt; LastErrors</a>
<a href="../help/HelpSystem.cs.html#334"><b>334</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e, &quot;HelpNotFound&quot;, ErrorCategory.ResourceUnavailable, null);</a>
<a href="../help/HelpSystem.cs.html#706"><b>706</b>internal void TraceError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../help/HelpSystem.cs.html#719"><b>719</b>internal void TraceErrors(Collection&lt;<i>ErrorRecord</i>&gt; errorRecords)</a>
</div>
<div class="rF"><div class="rN">help\MamlNode.cs (4)</div>
<a href="../help/MamlNode.cs.html#676"><b>676</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new ParentContainsErrorRecordException(&quot;MamlInvalidChildNodeError&quot;), &quot;MamlInvalidChildNodeError&quot;, ErrorCategory.SyntaxError, null);</a>
<a href="../help/MamlNode.cs.html#689"><b>689</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new ParentContainsErrorRecordException(&quot;MamlInvalidChildNodeCountError&quot;), &quot;MamlInvalidChildNodeCountError&quot;, ErrorCategory.SyntaxError, null);</a>
<a href="../help/MamlNode.cs.html#1248"><b>1248</b>internal Collection&lt;<i>ErrorRecord</i>&gt; Errors { get; } = new Collection&lt;<i>ErrorRecord</i>&gt;();</a>
</div>
<div class="rF"><div class="rN">help\ProviderHelpProvider.cs (1)</div>
<a href="../help/ProviderHelpProvider.cs.html#84"><b>84</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e, &quot;ProviderLoadError&quot;, ErrorCategory.ResourceUnavailable, null);</a>
</div>
<div class="rF"><div class="rN">help\SaveHelpCommand.cs (1)</div>
<a href="../help/SaveHelpCommand.cs.html#119"><b>119</b><i>ErrorRecord</i> error = new ErrorRecord(new InvalidOperationException(errMsg), &quot;ModuleAndFullyQualifiedModuleCannotBeSpecifiedTogether&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">help\UpdatableHelpCommandBase.cs (1)</div>
<a href="../help/UpdatableHelpCommandBase.cs.html#624"><b>624</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new Exception(errorMessage),</a>
</div>
<div class="rF"><div class="rN">help\UpdatableHelpSystem.cs (1)</div>
<a href="../help/UpdatableHelpSystem.cs.html#114"><b>114</b>internal <i>ErrorRecord</i> CreateErrorRecord(UpdatableHelpCommandType commandType)</a>
</div>
<div class="rF"><div class="rN">help\UpdateHelpCommand.cs (1)</div>
<a href="../help/UpdateHelpCommand.cs.html#170"><b>170</b><i>ErrorRecord</i> error = new ErrorRecord(new InvalidOperationException(errMsg), &quot;ModuleAndFullyQualifiedModuleCannotBeSpecifiedTogether&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">logging\LogProvider.cs (1)</div>
<a href="../logging/LogProvider.cs.html#176"><b>176</b><i>ErrorRecord</i> er = ier.ErrorRecord;</a>
</div>
<div class="rF"><div class="rN">namespaces\CoreCommandContext.cs (8)</div>
<a href="../namespaces/CoreCommandContext.cs.html#342"><b>342</b>private Collection&lt;<i>ErrorRecord</i>&gt; _accumulatedErrorObjects = new Collection&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../namespaces/CoreCommandContext.cs.html#892"><b>892</b>internal Collection&lt;<i>ErrorRecord</i>&gt; GetAccumulatedErrorObjects()</a>
<a href="../namespaces/CoreCommandContext.cs.html#896"><b>896</b>Collection&lt;<i>ErrorRecord</i>&gt; results = _accumulatedErrorObjects;</a>
<a href="../namespaces/CoreCommandContext.cs.html#897"><b>897</b>_accumulatedErrorObjects = new Collection&lt;<i>ErrorRecord</i>&gt;();</a>
<a href="../namespaces/CoreCommandContext.cs.html#937"><b>937</b>Collection&lt;<i>ErrorRecord</i>&gt; errors = GetAccumulatedErrorObjects();</a>
<a href="../namespaces/CoreCommandContext.cs.html#992"><b>992</b>foreach (<i>ErrorRecord</i> errorRecord in GetAccumulatedErrorObjects())</a>
<a href="../namespaces/CoreCommandContext.cs.html#1087"><b>1087</b>internal void WriteError(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">namespaces\DriveProviderBase.cs (1)</div>
<a href="../namespaces/DriveProviderBase.cs.html#148"><b>148</b>/// Any failures should be sent to the &lt;see cref=&quot;System.Management.Automation.Provider.CmdletProvider.WriteError(<i>ErrorRecord</i>)&quot;/&gt;</a>
</div>
<div class="rF"><div class="rN">namespaces\FileSystemProvider.cs (15)</div>
<a href="../namespaces/FileSystemProvider.cs.html#523"><b>523</b><i>ErrorRecord</i> er = new ErrorRecord(new NotSupportedException(FileSystemProviderStrings.PersistNotSupported), &quot;DriveRootNotNetworkPath&quot;, ErrorCategory.InvalidArgument, drive);</a>
<a href="../namespaces/FileSystemProvider.cs.html#635"><b>635</b><i>ErrorRecord</i> er = new ErrorRecord(new InvalidOperationException(FileSystemProviderStrings.InvalidDriveName), &quot;DriveNameNotSupportedForPersistence&quot;, ErrorCategory.InvalidOperation, drive);</a>
<a href="../namespaces/FileSystemProvider.cs.html#676"><b>676</b><i>ErrorRecord</i> er = new ErrorRecord(new System.ComponentModel.Win32Exception(code), &quot;CouldNotMapNetworkDrive&quot;, ErrorCategory.InvalidOperation, drive);</a>
<a href="../namespaces/FileSystemProvider.cs.html#771"><b>771</b><i>ErrorRecord</i> er = new ErrorRecord(new System.ComponentModel.Win32Exception(code), &quot;CouldRemoveNetworkDrive&quot;, ErrorCategory.InvalidOperation, drive);</a>
<a href="../namespaces/FileSystemProvider.cs.html#1363"><b>1363</b><i>ErrorRecord</i> er = new ErrorRecord(ioError, &quot;GetItemIOError&quot;, ErrorCategory.ReadError, path);</a>
<a href="../namespaces/FileSystemProvider.cs.html#2814"><b>2814</b><i>ErrorRecord</i> error = null;</a>
<a href="../namespaces/FileSystemProvider.cs.html#3295"><b>3295</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(e, &quot;RemoveFileSystemItemUnAuthorizedAccess&quot;, ErrorCategory.PermissionDenied, fileSystemInfo);</a>
<a href="../namespaces/FileSystemProvider.cs.html#3334"><b>3334</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(fsException, &quot;RemoveFileSystemItemUnAuthorizedAccess&quot;, ErrorCategory.PermissionDenied, fileSystemInfo);</a>
<a href="../namespaces/FileSystemProvider.cs.html#3341"><b>3341</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(fsException, &quot;RemoveFileSystemItemArgumentError&quot;, ErrorCategory.InvalidArgument, fileSystemInfo);</a>
<a href="../namespaces/FileSystemProvider.cs.html#3350"><b>3350</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(fsException, &quot;RemoveFileSystemItemIOError&quot;, ErrorCategory.WriteError, fileSystemInfo);</a>
<a href="../namespaces/FileSystemProvider.cs.html#3385"><b>3385</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(attributeException, &quot;RemoveFileSystemItemCannotRestoreAttributes&quot;, ErrorCategory.PermissionDenied, fileSystemInfo);</a>
<a href="../namespaces/FileSystemProvider.cs.html#3415"><b>3415</b><i>ErrorRecord</i> error = null;</a>
<a href="../namespaces/FileSystemProvider.cs.html#3443"><b>3443</b>private bool ItemExists(string path, out <i>ErrorRecord</i> error)</a>
<a href="../namespaces/FileSystemProvider.cs.html#6695"><b>6695</b><i>ErrorRecord</i> error = new ErrorRecord(new InvalidOperationException(errMsg), &quot;GetContainerContentException&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../namespaces/FileSystemProvider.cs.html#6857"><b>6857</b><i>ErrorRecord</i> error = new ErrorRecord(new InvalidOperationException(errMsg), &quot;WriteContainerContentException&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">namespaces\FileSystemSecurity.cs (2)</div>
<a href="../namespaces/FileSystemSecurity.cs.html#324"><b>324</b>private static <i>ErrorRecord</i> CreateErrorRecord(string path,</a>
<a href="../namespaces/FileSystemSecurity.cs.html#331"><b>331</b><i>ErrorRecord</i> er =</a>
</div>
<div class="rF"><div class="rN">namespaces\LocationGlobber.cs (4)</div>
<a href="../namespaces/LocationGlobber.cs.html#869"><b>869</b><i>ErrorRecord</i> errorRecord = context.GetAccumulatedErrorObjects()[0];</a>
<a href="../namespaces/LocationGlobber.cs.html#1073"><b>1073</b>Collection&lt;<i>ErrorRecord</i>&gt; errors = context.GetAccumulatedErrorObjects();</a>
<a href="../namespaces/LocationGlobber.cs.html#4279"><b>4279</b>Collection&lt;<i>ErrorRecord</i>&gt; errors = getChildNamesContext.GetAccumulatedErrorObjects();</a>
<a href="../namespaces/LocationGlobber.cs.html#4284"><b>4284</b>foreach (<i>ErrorRecord</i> errorRecord in errors)</a>
</div>
<div class="rF"><div class="rN">namespaces\ProviderBase.cs (2)</div>
<a href="../namespaces/ProviderBase.cs.html#1423"><b>1423</b>public void ThrowTerminatingError(<i>ErrorRecord</i> errorRecord)</a>
<a href="../namespaces/ProviderBase.cs.html#1951"><b>1951</b>public void WriteError(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">namespaces\RegistryProvider.cs (1)</div>
<a href="../namespaces/RegistryProvider.cs.html#1041"><b>1041</b><i>ErrorRecord</i> rec = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">security\CatalogHelper.cs (16)</div>
<a href="../security/CatalogHelper.cs.html#111"><b>111</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(exception, &quot;UnKnownCatalogVersion&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#142"><b>142</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(exception, &quot;UnKnownCatalogVersion&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#241"><b>241</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.FoundDuplicateFilesRelativePath, relativePath)), &quot;FoundDuplicateFilesRelativePath&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#318"><b>318</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(CatalogStrings.UnableToOpenCatalogDefinitionFile), &quot;UnableToOpenCatalogDefinitionFile&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#349"><b>349</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(CatalogStrings.CatalogDefinitionFileNotGenerated), &quot;CatalogDefinitionFileNotGenerated&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#408"><b>408</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.UnableToAcquireHashAlgorithmContext, hashAlgorithm)), &quot;UnableToAcquireHashAlgorithmContext&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#428"><b>428</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.UnableToCreateFileHash, filePath)), &quot;UnableToCreateFileHash&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#439"><b>439</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.UnableToCreateFileHash, filePath)), &quot;UnableToCreateFileHash&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#464"><b>464</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.UnableToReadFileToHash, filePath)), &quot;UnableToReadFileToHash&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#546"><b>546</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.UnableToOpenCatalogFile, catalogFilePath)), &quot;UnableToOpenCatalogFile&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#562"><b>562</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.UnableToOpenCatalogFile, catalogFilePath)), &quot;UnableToOpenCatalogFile&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#635"><b>635</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.FoundDuplicateFilesRelativePath, relativePath)), &quot;FoundDuplicateFilesRelativePath&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#802"><b>802</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.UnableToFindFileNameOrPathForCatalogMember, pwszLine)), &quot;UnableToFindFileNameOrPathForCatalogMember&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#808"><b>808</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.UnableToCreateFileHash, pwszLine)), &quot;UnableToCreateFileHash&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#814"><b>814</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.UnableToFindFileToHash, pwszLine)), &quot;UnableToFindFileToHash&quot;, ErrorCategory.InvalidOperation, null);</a>
<a href="../security/CatalogHelper.cs.html#824"><b>824</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(new InvalidOperationException(StringUtil.Format(CatalogStrings.FoundDuplicateFileMemberInCatalog, pwszLine)), &quot;FoundDuplicateFileMemberInCatalog&quot;, ErrorCategory.InvalidOperation, null);</a>
</div>
<div class="rF"><div class="rN">security\SecuritySupport.cs (6)</div>
<a href="../security/SecuritySupport.cs.html#866"><b>866</b>internal static string Encrypt(byte[] contentBytes, CmsMessageRecipient[] recipients, SessionState sessionState, out <i>ErrorRecord</i> error)</a>
<a href="../security/SecuritySupport.cs.html#1027"><b>1027</b>public void Resolve(SessionState sessionState, ResolutionPurpose purpose, out <i>ErrorRecord</i> error)</a>
<a href="../security/SecuritySupport.cs.html#1085"><b>1085</b>private void ResolveFromBase64Encoding(ResolutionPurpose purpose, out <i>ErrorRecord</i> error)</a>
<a href="../security/SecuritySupport.cs.html#1124"><b>1124</b>private void ResolveFromPath(SessionState sessionState, ResolutionPurpose purpose, out <i>ErrorRecord</i> error)</a>
<a href="../security/SecuritySupport.cs.html#1202"><b>1202</b>private void ResolveFromStoreById(ResolutionPurpose purpose, out <i>ErrorRecord</i> error)</a>
<a href="../security/SecuritySupport.cs.html#1246"><b>1246</b>private void ProcessResolvedCertificates(ResolutionPurpose purpose, X509Certificate2Collection certificatesToProcess, out <i>ErrorRecord</i> error)</a>
</div>
<div class="rF"><div class="rN">singleshell\config\MshConsoleLoadException.cs (2)</div>
<a href="../singleshell/config/MshConsoleLoadException.cs.html#52"><b>52</b>private <i>ErrorRecord</i> _errorRecord;</a>
<a href="../singleshell/config/MshConsoleLoadException.cs.html#61"><b>61</b>public <i>ErrorRecord</i> ErrorRecord</a>
</div>
<div class="rF"><div class="rN">singleshell\config\MshSnapinLoadException.cs (2)</div>
<a href="../singleshell/config/MshSnapinLoadException.cs.html#121"><b>121</b>private <i>ErrorRecord</i> _errorRecord;</a>
<a href="../singleshell/config/MshSnapinLoadException.cs.html#131"><b>131</b>public override <i>ErrorRecord</i> ErrorRecord</a>
</div>
<div class="rF"><div class="rN">utils\CommandDiscoveryExceptions.cs (3)</div>
<a href="../utils/CommandDiscoveryExceptions.cs.html#120"><b>120</b>public override <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/CommandDiscoveryExceptions.cs.html#137"><b>137</b>private <i>ErrorRecord</i> _errorRecord;</a>
<a href="../utils/CommandDiscoveryExceptions.cs.html#301"><b>301</b><i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">utils\EncodingUtils.cs (1)</div>
<a href="../utils/EncodingUtils.cs.html#79"><b>79</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">utils\ExecutionExceptions.cs (16)</div>
<a href="../utils/ExecutionExceptions.cs.html#30"><b>30</b>internal CmdletInvocationException(<i>ErrorRecord</i> errorRecord)</a>
<a href="../utils/ExecutionExceptions.cs.html#131"><b>131</b>_errorRecord = (<i>ErrorRecord</i>)info.GetValue(&quot;ErrorRecord&quot;, typeof(<i>ErrorRecord</i>));</a>
<a href="../utils/ExecutionExceptions.cs.html#160"><b>160</b>public override <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/ExecutionExceptions.cs.html#177"><b>177</b>private <i>ErrorRecord</i> _errorRecord = null;</a>
<a href="../utils/ExecutionExceptions.cs.html#467"><b>467</b>internal ActionPreferenceStopException(<i>ErrorRecord</i> error)</a>
<a href="../utils/ExecutionExceptions.cs.html#494"><b>494</b><i>ErrorRecord</i> errorRecord,</a>
<a href="../utils/ExecutionExceptions.cs.html#521"><b>521</b>_errorRecord = (<i>ErrorRecord</i>)info.GetValue(&quot;ErrorRecord&quot;, typeof(<i>ErrorRecord</i>));</a>
<a href="../utils/ExecutionExceptions.cs.html#609"><b>609</b>public override <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/ExecutionExceptions.cs.html#614"><b>614</b>private readonly <i>ErrorRecord</i> _errorRecord = null;</a>
<a href="../utils/ExecutionExceptions.cs.html#746"><b>746</b>/// &lt;see cref=&quot;System.Management.Automation.<i>ErrorRecord</i>.TargetObject&quot;/&gt;</a>
<a href="../utils/ExecutionExceptions.cs.html#890"><b>890</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/ExecutionExceptions.cs.html#907"><b>907</b>private <i>ErrorRecord</i> _errorRecord = null;</a>
<a href="../utils/ExecutionExceptions.cs.html#998"><b>998</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/ExecutionExceptions.cs.html#1015"><b>1015</b>private <i>ErrorRecord</i> _errorRecord = null;</a>
</div>
<div class="rF"><div class="rN">utils\GraphicalHostReflectionWrapper.cs (1)</div>
<a href="../utils/GraphicalHostReflectionWrapper.cs.html#80"><b>80</b><i>ErrorRecord</i> error = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">utils\MshArgumentException.cs (2)</div>
<a href="../utils/MshArgumentException.cs.html#120"><b>120</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/MshArgumentException.cs.html#137"><b>137</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
<div class="rF"><div class="rN">utils\MshArgumentNullException.cs (2)</div>
<a href="../utils/MshArgumentNullException.cs.html#118"><b>118</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/MshArgumentNullException.cs.html#135"><b>135</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
<div class="rF"><div class="rN">utils\MshArgumentOutOfRangeException.cs (2)</div>
<a href="../utils/MshArgumentOutOfRangeException.cs.html#116"><b>116</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/MshArgumentOutOfRangeException.cs.html#133"><b>133</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
<div class="rF"><div class="rN">utils\MshInvalidOperationException.cs (2)</div>
<a href="../utils/MshInvalidOperationException.cs.html#114"><b>114</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/MshInvalidOperationException.cs.html#131"><b>131</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
<div class="rF"><div class="rN">utils\MshNotImplementedException.cs (2)</div>
<a href="../utils/MshNotImplementedException.cs.html#97"><b>97</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/MshNotImplementedException.cs.html#114"><b>114</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
<div class="rF"><div class="rN">utils\MshNotSupportedException.cs (2)</div>
<a href="../utils/MshNotSupportedException.cs.html#97"><b>97</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/MshNotSupportedException.cs.html#114"><b>114</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
<div class="rF"><div class="rN">utils\MshObjectDisposedException.cs (2)</div>
<a href="../utils/MshObjectDisposedException.cs.html#103"><b>103</b>public <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/MshObjectDisposedException.cs.html#120"><b>120</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
<div class="rF"><div class="rN">utils\PathUtils.cs (7)</div>
<a href="../utils/PathUtils.cs.html#146"><b>146</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../utils/PathUtils.cs.html#180"><b>180</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../utils/PathUtils.cs.html#314"><b>314</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../utils/PathUtils.cs.html#328"><b>328</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../utils/PathUtils.cs.html#342"><b>342</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../utils/PathUtils.cs.html#383"><b>383</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
<a href="../utils/PathUtils.cs.html#404"><b>404</b><i>ErrorRecord</i> errorRecord = new ErrorRecord(</a>
</div>
<div class="rF"><div class="rN">utils\PowerShellETWTracer.cs (1)</div>
<a href="../utils/PowerShellETWTracer.cs.html#1002"><b>1002</b>public bool TraceErrorRecord(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">utils\RuntimeException.cs (6)</div>
<a href="../utils/RuntimeException.cs.html#102"><b>102</b><i>ErrorRecord</i> errorRecord)</a>
<a href="../utils/RuntimeException.cs.html#150"><b>150</b>public virtual <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/RuntimeException.cs.html#167"><b>167</b>private <i>ErrorRecord</i> _errorRecord;</a>
<a href="../utils/RuntimeException.cs.html#223"><b>223</b>internal static string RetrieveMessage(<i>ErrorRecord</i> errorRecord)</a>
<a href="../utils/RuntimeException.cs.html#245"><b>245</b><i>ErrorRecord</i> er = icer.ErrorRecord;</a>
<a href="../utils/RuntimeException.cs.html#255"><b>255</b>internal static Exception RetrieveException(<i>ErrorRecord</i> errorRecord)</a>
</div>
<div class="rF"><div class="rN">utils\SessionStateExceptions.cs (5)</div>
<a href="../utils/SessionStateExceptions.cs.html#94"><b>94</b>internal ProviderInvocationException(ProviderInfo provider, <i>ErrorRecord</i> errorRecord)</a>
<a href="../utils/SessionStateExceptions.cs.html#233"><b>233</b>public override <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/SessionStateExceptions.cs.html#251"><b>251</b>private <i>ErrorRecord</i> _errorRecord;</a>
<a href="../utils/SessionStateExceptions.cs.html#496"><b>496</b>public override <i>ErrorRecord</i> ErrorRecord</a>
<a href="../utils/SessionStateExceptions.cs.html#513"><b>513</b>private <i>ErrorRecord</i> _errorRecord;</a>
</div>
</div>
</body></html>